{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "004ddcf9",
   "metadata": {},
   "source": [
    "# Sepsis, Whole Blood Datasets:\n",
    "\n",
    "\n",
    "## GSE232753 \n",
    "\n",
    "Comparative gene expression profiling analysis of RNA-seq data for patients with sepsis and healthy volunteers. 20 sepsis , 8 controls.\n",
    "\n",
    "Sepsis + control . D0.\n",
    "\n",
    "Presented-genes: 6\n",
    "\n",
    "Missing-genes : 48\n",
    "\n",
    "Only 450 genes in expression analysis. \n",
    "\n",
    "There are lots of missing genes , so it is not appropriate for our analysis.\n",
    "\n",
    "## GSE63311\n",
    "\n",
    "For the RNA-Seq study reported here, 73 patients were recruited with deferred consent at the time of first examination in an emergency ward based on the opinion of physicians that there was a potential for the patient's condition to develop into sepsis. These were retrospectively divided into groups based on clinical features and compared to 11 non-urgent surgical controls.\n",
    "\n",
    "Sepsis + no sepsis surgical control+ control. \n",
    "\n",
    "Presented-genes: 48\n",
    "\n",
    "Missing-genes : 6(\"CALCA\"   \"LBP\"   \"IFNA\"   \"IFNB\"  \"CCL25\"   \"HLA-DR\")\n",
    "\n",
    "Some aliases have been found .\n",
    "\n",
    "Missing genes 3 : \"CALCA\"   \"LBP\"  \"CCL25\"\n",
    "\n",
    "Also searched by ensembl gene id.\n",
    "\n",
    "## GSE28750 \n",
    "\n",
    "This was a multi-centre, prospective clinical trial conducted across 4 tertiary critical care settings in Australia. Sepsis patients were recruited if they met the 1992 Consensus Statement criteria and had clinical evidence of systemic infection based on microbiology diagnoses (n=27). Participants in the post-surgical (PS) group were recruited pre-operatively and blood samples collected within 24 hours following surgery (n=38). Healthy controls (HC) included hospital staff with no known concurrent illnesses (n=20). Each participant had minimally 5ml of PAXgene blood collected for leucocyte RNA isolation and gene expression analyses. The GEO data represents the preliminary microarray investigation performed on Human U133 Plus 2.0 GeneChips (Affymetrix), assaying 41 patient samples (Sepsis n=10, Post-Surgical n=11, Control n=20).\n",
    "\n",
    "Sepsis + post surgical + control. \n",
    "\n",
    "Presented-genes: 51\n",
    "\n",
    "Missing-genes : 3 (\"IFNA\"  \"IFNB\"  \"HLA-DR\")\n",
    "\n",
    "All genes are present by aliases .\n",
    "\n",
    "## GSE54514 \n",
    "\n",
    "Whole blood samples were collected daily for up to 5 days from patients admitted to the intensive care unit with sepsis. RNA isolated from whole blood samples was assayed on Illumina HT-12 gene expression microarrays consisting of 48,804 probes. Microarray analysis identified 3677 genes as differentially expressed across 5 days between septic patients and healthy controls.Daily PAXgene samples for up to 5 days for sepsis survivors (n=26), sepsis nonsurvivors (n=9), and healthy controls (n=18).\n",
    "\n",
    "Sepsis + control. (survivor, non survivor). D1/2/3/4/5\n",
    "\n",
    "Presented-genes: 46\n",
    "\n",
    "Missing-genes : 8 (\"CRP\"  \"LBP\"   \"ELANE\" \"CXCL8\" \"IFNA\"  \"IFNB\"  \"PTX3\"  \"HLA-DR\")\n",
    "\n",
    "Some aliases have been found IL6 as HGF.\n",
    "\n",
    "Missing genes 3 : \"CRP\"  \"LBP\"  \"PTX3\"\n",
    "\n",
    "## GSE242059 \n",
    "\n",
    "comprehensive Analysis of ceRNA Network Composed of circRNA, miRNA, and mRNA in septic acute kidney injury patients based on RNA-seq.We collected 10 samples of whole blood of adults, including 5 cases of SAKI and 5 cases of healthy controls.\n",
    "\n",
    "Septis acute kidney injury + healthy. \n",
    "\n",
    "miRNA datasets.\n",
    "\n",
    "\n",
    "## GSE69528\n",
    "\n",
    "Total RNA from whole blood obtained from patients with sepsis caused by B.pseudomallei (n=29) or other pathogens (n=54) and uninfected controls (28 healthy and 27 subjects with type 2 diabetes mellitus) were collected. In order to validate the published signature, microarray data were generated from these samples. This dataset was also used for an independent selection of signatures for septicemic melioidosis. The same RNA samples were used for validation by a high throughput real-time PCR technique, Fluidigm.\n",
    "\n",
    "Sepsis caused by different pathogens + healthy control. \n",
    "\n",
    "Presented-genes: 40\n",
    "\n",
    "Missing-genes:14 ( \"CALCA\"  \"CRP\"    \"LBP\"    \"IL6\"    \"CXCL8\"  \"IFNA\"   \"IFNB\"   \"CCL19\"  \"CCL25\"  \"PTX3\"   \"HLA-DR\" \"PDCD1\"  \"CCL2\"   \"VCAM1\")\n",
    "\n",
    "Some aliases have been found IL6 as HGF.\n",
    "\n",
    "Missing genes 9 :\"CALCA\"  \"CRP\"  \"LBP\"  \"CCL19\"  \"CCL25\"  \"PTX3\" \"PDCD1\"  \"CCL2\"   \"VCAM1\"\n",
    "\n",
    "\n",
    "## GSE13015\n",
    "\n",
    "Whole blood obtained from patients with sepsis caused by B.pseudomallei or other pathogens were collected. mRNA Expression level of each patient was determined and compared to mRNA expression level from uninfected control including patient who had recovered from melioidosis, patient diagnosed type 2 diabetes, and healthy donors using microarray analysis.\n",
    "\n",
    "Sepsis caused by diffrent pathogens + healthy controls. \n",
    "\n",
    "GLP6106: 48 genes present + 6 missing.(\"ELANE\" \"CXCL8\" \"IFNA\" \"IFNB\" \"HLA-DR\" \"NLRP3\")\n",
    "\n",
    "GLP6947: 49 genes present + 5 missing.(\"ELANE\" \"CXCL8\" \"IFNA\" \"IFNB\" \"HLA-DR\")\n",
    "\n",
    "\n",
    "GLP6106: All genes are present by their aliases. NLRP3 as (\"CIAS1\" \"AVP\")\n",
    "\n",
    "GLP6947: All genes are present by their aliases.\n",
    "\n",
    "## GSE208581\n",
    "\n",
    "Risk assessment with gene expression markers in sepsis development. Whole blood RNA sequencing analysis was performed on preoperative samples taken from 267 patients. These comprised patients who developed postoperatively infection leading to sepsis (n=77) or an uncomplicated infection outcome (n=49), non-infectious systemic inflammatory response (n=31), or an uncomplicated postoperative course (n=110).\n",
    "\n",
    "preoperative samples f sepsis + infection + … . \n",
    "\n",
    "Presented-genes: 49\n",
    "\n",
    "Missing-genes : 5 (S100A8   S100A9   IFNA    IFNB   HLA-DR).\n",
    "\n",
    "Some aliases have been found but  S100A8  and  S100A9 are missing.\n",
    "\n",
    "Searched also by ensemble id .By “MIF” name is present but it may not be the exact gene.\n",
    "\n",
    "## GSE224095\n",
    "\n",
    ",Single-cell transcriptomic changes in sepsis patients. Here we analyzed the whole blood transcriptome of 5 healthy controls and 5 patients with confirmed sepsis from the Department of Emergency, Division of Surgical Critical care, Tongji Trauma Center, Tongji Hospital, Tongji Medical College. This research aims to understand the expression of immune cells in sepsis patients and to construct an immune cell landscape of sepsis.\n",
    "\n",
    "Sepsis + control.\n",
    "\n",
    "2 samples, I couldn’t open the files properly. Crash. library(Seurat)\n",
    "\n",
    "## GSE185263\n",
    "\n",
    "Whole blood RNA-seq and clinical data was collected from 348 patients from four emergency rooms and one intensive care unit.Predicting sepsis severity at first clinical presentation: the role of endotypes and mechanistic signatures.\n",
    "\n",
    "Sepsis + healthy. (survivor, non survivor). Sofa score.\n",
    "\n",
    "Presented-genes: 51\n",
    "\n",
    "Missing-genes : 3 (IFNA   IFNB   HLA-DR )\n",
    "\n",
    "All genes are present by their aliases.\n",
    "\n",
    "\n",
    "\n",
    "## GSE32707\n",
    "\n",
    "A microarray analysis involving whole blood samples isolated from critically ill patients in the medical intensive care unit at Brigham and Women's Hospital. Four groups of intubated subjects undergoing mechanical ventilation were recruited for the study: those with sepsis alone (Sepsis), those with sepsis + ARDS (se/ARDS), those with SIRS (SIRS), and those without sepsis, SIRS, or ARDS (untreated). Blood was obtained from patients on the day of admission (day 0) and 7 days later. RNA was isolated from the whole blood samples and microarrays were prepared to determine differential gene expression between the four groups.\n",
    "\n",
    "Sepsis + sepsis/ARDS + SIRS + patient with non of them. D0/7.\n",
    "\n",
    "Presented-genes: 50\n",
    "\n",
    "Missing-genes : 4 ( CXCL8   IFNA   IFNB   HLA-DR)\n",
    "\n",
    "All genes are present by their aliases.\n",
    "\n",
    "## GSE211210\n",
    "A total of 10 patients with acute sepsis and 10 health donors were recruited to take the next-generation RNA-seq analysis and total RNA was purified from blood samples from sepsis patients (n = 5) and healthy control subjects (n = 5).\n",
    "\n",
    "Acute sepsis + healthy controls.\n",
    "\n",
    "Presented-genes: 51\n",
    "\n",
    "Missing-genes : 3 (IFNA   IFNB    HLA-DR)\n",
    "\n",
    "All genes are present by their aliases.\n",
    "\n",
    "\n",
    "## GSE100159\n",
    "A Transcriptome Fingerprinting Assay for Clinical Immune Monitoring (Sepsis Datasets).Sepsis (35 condition, 12 control, 47 total).\n",
    "\n",
    "Sepsis caused infections + controls.\n",
    "\n",
    "Presented-genes: 49\n",
    "\n",
    "Missing-genes : 5( ELANE   CXCL8    IFNA   IFNB   HLA-DR)\n",
    "\n",
    "All genes are present by their aliases.\n",
    "\n",
    "## GSE60424\n",
    "\n",
    "We performed whole genome RNAseq profiling of immune cell subsets and whole blood from subjects with an array of immune-associated diseases.This study compared whole transcriptome signatures of 6 immune cell subsets and whole blood from patients with an array of immune-associated diseases. Fresh blood samples were collected from healthy subjects and subjects diagnosed type 1 diabetes, amyotrophic lateral sclerosis, and sepsis, as well as multiple sclerosis patients before and 24 hours after the first treatment with IFN-beta.\n",
    "\n",
    "Sepsis + diabetes1 + ALS + healthy . whole blood and other (B cell, monocytes, …)\n",
    "\n",
    "Presented-genes: 51\n",
    "\n",
    "Missing-genes : 3 (IFNA    IFNB    HLA-DR)\n",
    "\n",
    "All genes are present by their aliases.\n",
    "\n",
    "\n",
    "## GSE29536 \n",
    "\n",
    "Whole Blood Transcriptional Modules generated on Illumina Hu-6 V2 Beadchips.This series combines nine independent datasets representing a spectrum of human pathologies expected to result in changes in gene abundance related to changes in expression or cellular composition of whole blood. These nine datasets are composed of 410 individual whole blood profiles generated from patients with HIV, tuberculosis, sepsis, systemic lupus erythematosus, systemic arthritis, B-cell deficiency and liver transplant. For each dataset healthy controls are also included.\n",
    "\n",
    "Sepsis + HIV + tuberculosis + .. .+ healthy control.\n",
    "\n",
    "Presented-genes: 47\n",
    "\n",
    "Missing-genes : 7 (\"HMGB1\"  \"ELANE\"  \"CXCL8\"  \"FCGR1A\" \"IFNA\"   \"IFNB\"   \"HLA-DR\")\n",
    "\n",
    "Some aliases have been found but  \"HMGB1\" and \"FCGR1A\" are missing.\n",
    "\n",
    "\n",
    "## GSE222393\n",
    "\n",
    "Gene expression trajectories were compared between 20 severe COVID-19 patients and 22 contemporaneous non-COVID-19 sepsis patients in the intensive care unit (ICU). Whole blood was collected at two timepoints: ICU admission and ~1 week later.\n",
    "\n",
    "Sepsis + sepsis and COVID. (survivor, non survivor).\n",
    "\n",
    "Presented-genes: 51\n",
    "\n",
    "Missing-genes : 3 (IFNA    IFNB   HLA-DR)\n",
    "\n",
    "Raw data was separated for individuals, inspecting just one sample.\n",
    "\n",
    "All genes are present by their aliases.\n",
    "\n",
    "\n",
    "## GSE217700\n",
    "LncRNA and mRNA profiles of 4 sepsis patients whole blood samples and 4 healthy controls whole blood samples.\n",
    "\n",
    "Sepsis + healthy. \n",
    "\n",
    "Presented-genes: 51\n",
    "\n",
    "Missing-genes : 3( IFNA    IFNB   HLA-DR)\n",
    "\n",
    "All genes are present by their aliases.\n",
    "\n",
    "\n",
    "\n",
    "## GSE137340\n",
    "\n",
    "Transcriptomic profiling was performed on sepsis cases and on matched (by age and gender) healthy controls, who were not suffering from any inflammatory diseases, and were not related to the patients. Samples were collected at two different time points in the cases. Two blood samples from the sepsis cases were collected, the first at the time of diagnosis (D1), and a second sample after 24 hours (D2) . Representative single blood sample was collected from each of the healthy control individuals. A total of 27 patients (23 with paired transcriptome data for two time points) and 12 healthy control subjects (for the single time point) were recruited in this study.\n",
    "\n",
    "Sepsis + severe sepsis +septic shock + healthy.(survivor, non survivor). D1/2\n",
    "\n",
    "Presented-genes: 52\n",
    "\n",
    "Missing-genes : 3( PTX3   MMP8)\n",
    "\n",
    "\n",
    "\n",
    "## GSE137342\n",
    "\n",
    "same as the dataset GSE137340. Same samples.\n",
    "\n",
    "\n",
    "## SE100150\n",
    "\n",
    "Total of 985 samples across 16 datasets. Each dataset contains it's own biological controls, there are no replicates. Samples are distributed as follows: TB (23 condition, 11 control, 34 total); Staph aureus (99 condition, 44 control, 143 total); Sepsis (35 condition, 12 control, 47 total); HIV (28 condition, 35 control, 63 total); Flu (25 condition, 14 control, 39 total); RSV (70 condition, 14 control, 84 total); B-cell deficiency (20 condition, 13 control, 33 total); Liver Transplant (94 condition, 30 control, 124 total); Pregnancy (25 condition, 20 control, 45 total); Melanoma Stage IV (22 condition, 5 control, 27 total); Kawasaki (21 condition, 23 control, 44 total); Juvenile Dermatomyositis (40 condition, 9 control, 49 total); COPD (19 condition, 24 control, 43 total); MS - untreated (34 condition, 22 control, 56 total); Pediatric SLE (55 condition, 14 control, 69 total); SoJIA (62 condition, 23 control, 85 total)\n",
    "\n",
    "Sepsis + 15 other immune disease + controls. \n",
    "\n",
    "Presented-genes: 49\n",
    "\n",
    "Missing-genes : 5(ELANE    CXCL8    IFNA     IFNB    HLA-DR)\n",
    "\n",
    "All genes are present by their aliases.\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.16"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
