{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "32866f6e",
   "metadata": {},
   "source": [
    "# peripheral blood mononuclear cells (PBMCs): \n",
    "\n",
    "\n",
    "## GSE241238 \n",
    "\n",
    "analyzes and compares the test transcripts of peripheral blood mononuclear cells (PBMCs) isolated from 4 healthy donors and 4 septic patients' whole blood.\n",
    "\n",
    "Sepsis + helthy control. \n",
    "\n",
    "Presented-genes: 54\n",
    "\n",
    "Missing-genes :0\n",
    "\n",
    "All the genes of interests are present.\n",
    "\n",
    "\n",
    "## GSE5271\n",
    "\n",
    "Time evolution study of sepsis in patients at D1, D7 and D28 post-inclusion. The day of inclusion corresponded to the first blood sampling (15 ml) quoted day 0. The following blood samples were performed at days 1, 7, and 28. Since the pre-sepsis expression of the genes as a control cannot be obtained, the day 0 sample was used patient per patient as a reference value for the next day's samples for microarray study. Patients' evolution was considered.\n",
    "Sepsis patients. D1/7/28. Focused only on 340 inflammatory genes.  ONLY 340 GENES.\n",
    "\n",
    "Present genes :13\n",
    "\n",
    "Missing genes: 41\n",
    "\n",
    "Search for all aliases and find 2 more genes:\n",
    "CCL2 as \"SCYA2\" and CCL25 as \"SCYA25\"\n",
    "\n",
    "\n",
    "## GSE199816\n",
    "\n",
    "Transcriptional Differences Between COVID-19 and Sepsis. Comparing the mRNA levels of whole blood samples from COVID-19 patients with septic patients clarified a pathology that is specific to COVID-19.\n",
    "\n",
    "Sepsis + COVID patients.\n",
    "\n",
    "RAW data is available, just investigate one sample of sepsis patients.\n",
    "\n",
    "Present genes :54\n",
    "\n",
    "Missing genes: 0\n",
    "\n",
    "All the genes of interest are present.\n",
    "\n",
    "\n",
    "## GSE216902\n",
    "\n",
    "We enrolled 37 elderly patients with sepsis at a tertiary referral hospital in central Taiwan. Blood samples and clinical/laboratory data were collected on day-1 and day-8 for RNA-Seq analysis. We implemented the DVID enrichment analysis and Gene Set Enrichment Analysis to identify the dynamic transcriptome and enriched pathway of day-8/day-1 differentially expressed genes, respectively. Finally, we utilized the MiXCR to analyze the diversity of T cell repertoire.Whole transcriptome analysis of critically ill elderly patients with sepsis.\n",
    "Sepsis patients. Responder and non responder D1/8. \n",
    "\n",
    "Present genes :54\n",
    "\n",
    "Missing genes: 0\n",
    "\n",
    "All genes of interest are present in the dataset.\n",
    "\n",
    "\n",
    "## GSE243217\n",
    "\n",
    "We performed the single RNA-seq analysis using whole blood from 22 sepsis patients and 35 COVID-19 patients admitted to our department, as well as 15 healthy subjects.GSE243218\n",
    "\n",
    "Sepsis + COVID + healthy . \n",
    "\n",
    "Raw data was available for each sample. One sample from a sepsis patient. \n",
    "\n",
    "Present genes :54\n",
    "\n",
    "Missing genes: 0\n",
    "\n",
    "All genes of interest are present in the dataset.\n",
    "\n",
    "\n",
    "## GSE109704\n",
    "\n",
    "The study population consisted of 30 melioidosis cases, 10 healthy controls and 10 sepsis cases caused by other pathogens. Total RNA was extracted from peripheral blood mononuclear cells (PBMCâ€™s) of study subjects. Gene expression profiles of 25 gene targets including 19 immune response genes and 6 epigenetic factors were analyzed by real time quantitative polymerase chain reaction (RT-qPCR).\n",
    "\n",
    "Sepsis patients + melioidosis + healthy control. 25 preselected genes.\n",
    "\n",
    "Present genes :4\n",
    "\n",
    "Missing genes: 51\n",
    "\n",
    "\n",
    "## GSE49754\n",
    "\n",
    "PBMCs were isolated from a healthy donor. Plasma samples were obtained from a first set of patients with culture-confirmed sepsis (n=29) and from uninfected controls (n=17). PBMCs were cultured for 6 h in medium alone, plasma from patients with sepsis, plasma from uninfected controls, and LPS using a final concentration of 20%. Transcriptional profiles were acquired using Illumina HumanHT12 V4 BeadChips.\n",
    "Medium alone (control).\n",
    "Plasma from sepsis patients.\n",
    "Plasma from uninfected controls.\n",
    "Lipopolysaccharide (LPS) as a positive control for immune activation.\n",
    "\n",
    "Present genes :49\n",
    "\n",
    "Missing genes: 6(\"CRP\"   \"LBP\"   \"IFNA1\" \"IFNA2\" \"IFNB1\" \"CD177\")\n",
    "\n",
    "\n",
    "## GSE175453\n",
    "\n",
    "A mixture of whole blood myeloid-enriched and Ficoll-enriched peripheral blood mononuclear cells from four late septic patients (post-sepsis day 14-21, RAP n=1 and CCI n=3) and five healthy subjects underwent Cellular Indexing of Transcriptomes and Epitopes by Sequencing (CITE-seq).\n",
    "unique transcriptomic patterns in non-myeloid immune cells during late sepsis.D14-21.\n",
    "\n",
    "Septic patients(single-cell RNA sequencing).\n",
    " \n",
    "## SCP548\n",
    "\n",
    "dataset focused on sepsis. It comprises single-cell RNA sequencing (scRNA-seq) data from peripheral blood mononuclear cells (PBMCs) of individuals diagnosed with bacterial sepsis. The dataset includes samples from 29 sepsis patients and 36 control subjects across three clinical cohorts.Single Cell Portal\n",
    "\n",
    "Sepsis + control.\n",
    "\n",
    "Single cell RNA-seq.\n",
    "\n",
    "## GSE182522\n",
    "\n",
    "A case-control study. A total of 18 immunocompromised subjects with sepsis and 18 sequential organ failure assessment (SOFA) score-matched immunocompetent control subjects were enrolled.\n",
    "\n",
    "Sepsis + SOFA + control. \n",
    "\n",
    "Present genes :54\n",
    "\n",
    "Missing genes: 0\n",
    "\n",
    "All genes of interest are present in the dataset.\n",
    "\n",
    "\n",
    "## GSE48080\n",
    "\n",
    "In this study we investigate whole-genome gene expression profiles of mononuclear cells from survivor (n=5) and non-survivor (n=5) septic patients, as well as from 3 healthy controls. Blood samples were collected at the time of sepsis diagnosis and seven days later, allowing us to evaluate the role of biological processes or genes possibly involved in patient recovery. Aiming to circumvent, at least partially, the heterogeneity of septic patients we included only patients admitted with sepsis caused by community-acquired pneumonia.\n",
    "\n",
    "Severe sepsis +septic shock + healthy control .(survivor, non survivor).\n",
    "\n",
    "Present genes :52\n",
    "\n",
    "Missing genes: 2 (\"FCGR1A\" \"IFNA1\")\n",
    "\n",
    "\n",
    "## GSE9960\n",
    "\n",
    "Gene-expression profiling of peripheral blood mononuclear cells in sepsis.To identify signature genes that help distinguish (1) sepsis from non-infectious causes of systemic inflammatory response syndrome, (2) between Gram-positive and Gram-negative sepsis.A total of 70 critically ill patients were enrolled in a single-centre observational study. Gene-expression profiling was performed using Affymetrix microarray (U133plus2) with 54,675 transcripts. Data was divided into a training set (n=35) and a validation set (n=35). A molecular signature was developed in the training set and was then validated in the validation set.\n",
    "\n",
    "Sepsis caused by different pathogens(g+/-) + healthy controls.\n",
    "\n",
    "Present genes :54\n",
    "\n",
    "Missing genes: 0\n",
    "\n",
    "All the genes of interest are present in dataset.\n",
    "\n",
    "## GSE167363\n",
    "\n",
    "Single-cell RNA-sequencing of human peripheral blood mononuclear cells from healthy controls, survivor and non-survivor of gram-negative sepsis patients.Dynamic changes in human single cell transcriptional signatures during fatal sepsis.\n",
    "\n",
    "G- sepsis patients + healthy control.(survivor, non survivor).\n",
    "\n",
    "\n",
    "\n",
    "## GSE167914 \n",
    "\n",
    "Impaired peripheral mononuclear cell metabolism in patients at risk of developing sepsis.Mitochondrial respiration and gene expression related to mitochondrial function were measured from the peripheral blood of infection and sepsis patients as well as healthy controls.gene expression related to mitochondrial function in PBMCs.\n",
    "\n",
    "Sepsis + control . (survivor, non survivor). Few genes.\n",
    "\n",
    "Present genes :1(BCL2)\n",
    "\n",
    "Missing genes: 53\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.16"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
