{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "02b92889",
   "metadata": {},
   "source": [
    "# BLood  leukocytes:\n",
    "\n",
    "\n",
    "\n",
    "## GSE134347 \n",
    "\n",
    "Whole-blood transcriptional patterns of protein-coding and non-coding RNA molecules in healthy subjects, critically-ill patients with sepsis or non-infectious conditions were generated by means of the human transcriptome array 2.0 (HTA 2.0) chip. Whole-blood from patients was collected within 24 hours of intensive care unit (ICU) admission. Title legend: healthy, healthy subjects; sepsis, ICU patients with sepsis; noninfectious, ICU patients with non-infectious conditions. \n",
    "\n",
    "Sepsis + non infectious + healthy.\n",
    "\n",
    "Couldn't find the gene annotation and mutual column in expression data and gpl file.\n",
    "\n",
    "\n",
    "\n",
    "## GSE134358 \n",
    "\n",
    "Genome-wide gene expression profiling of whole blood leukocytes in critically ill patients with sepsis or non-infectious disease has been used extensively in search of diagnostic biomarkers, as well as prognostic signatures reflecting diseases severity and outcome.\n",
    "\n",
    "Sepsis + non infectious + healthy.\n",
    "\n",
    "miRNA  and snoRNA dataset. \n",
    "\n",
    "\n",
    "## GSE134364\n",
    "\n",
    "Protein-coding and non-coding RNA landscape in critically ill patients with sepsis.\n",
    "\n",
    "Sepsis + noninfectious + patients+ healthy. \n",
    "\n",
    "Couldn't find the gene annotation and mutual column in expression data and gpl file.\n",
    "\n",
    "\n",
    "## GSE236713\n",
    "\n",
    "Biomarker Validation on the ‘Analysis of geNe Expression and bioMarkers fOr poiNt-of-care dEcision support in Sepsis’ (ANEMONES) study. Blood samples were collected from sepsis and SIRS patients at day 1, day 2 and day 5 of admittance to an intensive care unit (ICU) and on discharge from ICU. Some time-points were not collected due to patient death, patients leaving ICU or events beyond our control. Healthy control blood samples were collected from volunteers once and were recorded as day 1 only.\n",
    "\n",
    "Sepsis + SIRs + controls. (survivor, non survivor). D1/2/5. \n",
    "\n",
    "Present genes :52\n",
    "\n",
    "Missing genes: 2 (\"FCGR1A\" \"IFNA1\")\n",
    "\n",
    "\n",
    "## GSE5772\n",
    "\n",
    "To examine whether gene-expression profiling of circulating leukocytes is different between patients with sepsis as compared to controls. The signature genes identified will then be used to develop a prediction model to assist the diagnosis of sepsis.\n",
    "\n",
    "Sepsis patients + control patients. \n",
    "\n",
    "Couldn't find the annotation file\n",
    "\n",
    "\n",
    "\n",
    "## E-MTAB-5273,E-MTAB-5274\n",
    "\n",
    "We assayed leukocyte global gene expression for a prospective validation cohort of 221 adult patients admitted to UK intensive care units with sepsis due to community acquired pneumonia or faecal peritonitis. 10 samples from patients scheduled for elective cardiac surgery were also assayed as non-septic controls. We assigned all samples to sepsis response signature groups after performing unsupervised analysis of the transcriptomic data.\n",
    "Sepsis caused by Community-acquired pneumonia and Faecal peritonitis + non septic controls.\n",
    " (survivor, non survivor).\n",
    "\n",
    "\n",
    "E-MTAB-5273 : Present genes :47\n",
    "\n",
    "Missing genes: 7 (\"CALCA\" \"CRP\"   \"LBP\"   \"IFNB1\" \"CCL19\" \"PTX3\"  \"PDCD1\" \"CCL2\") \n",
    "\n",
    "E-MTAB-5274 : Present genes :47\n",
    "\n",
    "Missing genes: 7 (\"CALCA\" \"CRP\"   \"LBP\"   \"IFNB1\" \"CCL19\" \"PTX3\"  \"PDCD1\" \"CCL2\") \n",
    "\n",
    "\n",
    "## E-MTAB-4451, E-MTAB-4421 \n",
    "\n",
    "We assayed leukocyte global gene expression for a prospective validation cohort of 106 adult patients admitted to UK intensive care units with severe sepsis due to community acquired pneumonia.\n",
    "Sepsis CAP . (survivor, non survivor).\n",
    "\n",
    "E-MTAB-4451 : Present genes :44\n",
    "\n",
    "Missing genes:10 (\"CALCA\" \"CRP\"   \"LBP\"   \"IFNA1\" \"IFNA2\" \"IFNB1\" \"CCL19\" \"CCL25\" \"PTX3\"  \"PDCD1\" \"CCL2\"\n",
    ") \n",
    "\n",
    "E-MTAB-4421 : Present genes : can not download the phenotype data\n",
    "\n",
    "Missing genes: \n",
    "\n",
    "## GSE196117\n",
    "\n",
    "Examination of PAXgene blood leukocyte RNA expression in critically-ill patients due to sepsis treated with clarithromycin after the first day of study inclusion (INC_D1_A) and in the same patients after five days (INC_D5_A). Placebo-treated patients after one day of study inclusion (INC_D1_B) and same patients treated with placebo after five days (INC_D5_B). Healthy pariticpants were also included and assessed (healthy).\n",
    "\n",
    "Sepsis + treated sepsis + healthy control. D1/5.\n",
    "\n",
    "Present genes :54\n",
    "\n",
    "Missing genes: 0\n",
    "\n",
    "All the genes of interest are present in the dataset.\n",
    "\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.16"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
