{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "920172bb",
   "metadata": {},
   "source": [
    "# GSE185263 Dataset Description\n",
    "\n",
    "The dataset GSE185263 comprises gene expression profiles derived from whole blood RNA sequencing of **348 patients** across four emergency rooms and one intensive care unit. This dataset was designed to explore transcriptional changes in sepsis patients, assess disease severity, predict organ dysfunction, mortality, and uncover specific endotypes and mechanisms related to sepsis.\n",
    "\n",
    "\n",
    "### Data Preparation and Processing:\n",
    "\n",
    "Genes of interest have been identified and filtered based on their Ensembl Gene IDs and Gene Symbols. All genes of interest were present in the dataset and successfully searched using both Ensembl Gene IDs and Gene Symbols through BioMart.\n",
    "\n",
    "Presented-genes: 55\n",
    "\n",
    "Missing-genes :0\n",
    "\n",
    "### Normalization of raw count data\n",
    "\n",
    "The dataset consists of raw read counts for each gene across samples. These counts represent the number of sequencing reads mapped to each gene, which are not directly comparable between samples without normalization.\n",
    "\n",
    "**Variance Stabilization Normalization (VSN)**:The VSN method was applied to normalize the raw counts.\n",
    "VSN uses a variance stabilization transformation that makes the variance roughly constant across the range of mean intensities. This approach helps to reduce heteroscedasticity in the data, which can improve downstream statistical analyses.\n",
    "\n",
    "The process involves:\n",
    "\n",
    "Converting the raw counts into a numerical matrix, excluding metadata (e.g., Ensembl Gene IDs).\n",
    "\n",
    "Applying the justvsn function from the vsn R package to normalize the data.\n",
    "\n",
    "Verifying the normalization fit using the meanSdPlot function to ensure that the relationship between mean and standard deviation of gene expression is stabilized.\n",
    "\n",
    "### Two labeled datasets have been generated: (rows 392, cols 57)\n",
    "\n",
    "**Sepsis vs. Healthy Dataset**: Includes samples labeled as either Sepsis or Healthy Controls for sepsis diagnosis.(sepsis_labeled_dataGSE185263.csv).\n",
    "\n",
    "Sepsis = 348 —- healthy = 44.\n",
    "\n",
    "**Prognostic Dataset**: Includes samples labeled as Survived or Deceased for assessing the prognosis of sepsis patients.(sepsis_prognos_labeledGSE185263.csv)\n",
    "\n",
    "Survived = 293 —- died = 52 —- NA = 47.\n",
    "\n",
    "All the prepared files are in Dataset directory with corresponding GSE number.\n",
    "\n",
    "### Random forest\n",
    "\n",
    "The random forest test is done on (sepsis_labeled_dataGSE185263.csv) dataset. and the target label was cosidered as factor. (RFlabel-GSE185263code.R)\n",
    "\n",
    "here is the results:\n",
    "\n",
    "\n",
    "## average mettics results:\n",
    "\n",
    "After 100 repeated random forest by randomly splitting the data(repeated_splits_metrics.csv), ther average metrics was :\n",
    "\n",
    "\"Metric\",\"Value\"\n",
    "\n",
    "\"MCC\",0.79256891996504\n",
    "\n",
    "\"F1\",0.979106278871455\n",
    "\n",
    "\"AUC\",0.988940217391304\n",
    "\n",
    "\"TPR\",0.984492753623188\n",
    "\n",
    "\"TNR\",0.77125\n",
    "\n",
    "\"PPV\",0.973971493263954\n",
    "\n",
    "\"NPV\",0.863718253968254\n",
    "\n",
    "## Feature removal random forest:\n",
    "\n",
    "Shows by removing which column that is represnts one gene of interest the performance for prediction will drop significantly.(feature_removal_results.csv)\n",
    "\n",
    "**feature removing plot**:\n",
    "\n",
    "X-Axis: The values along the x-axis represent the performance difference ( using MCC ) when a specific feature is removed from the model.\n",
    "\n",
    "Y-Axis: The features (genes) are listed. Features at the top show a greater drop in performance upon removal, while features at the bottom show a smaller drop in performance.\n",
    "\n",
    "**Top Features (S100A9, IL10, FCGR1A, CD177, TL4)**:\n",
    "\n",
    "These are likely key biomarkers and should be prioritized in biological interpretation and downstream analysis.\n",
    "They are central to the model's predictions and could be explored further for biological relevance.\n",
    "\n",
    "**Middle Features (CCR7, PTX3, CALCA, ITGAM)**:\n",
    "\n",
    "These features are moderately important and could complement the top features.\n",
    "They might also contribute indirectly through feature interactions.\n",
    "\n",
    "**Bottom Features (MYD88, HLA.DRA, IFNG, TNFSF10, CCL2)**:\n",
    "\n",
    "These features might not be crucial for prediction in this context.\n",
    "Consider removing them to simplify the model or explore their redundancy with other features.\n",
    "\n",
    "## Sanity check\n",
    "\n",
    "Noise is artificially introduced into the dataset, and the model's performance metrics are observed as the noise level increases.\n",
    "\n",
    "Noise Level 0: Represents the original data with no added noise (baseline performance).\n",
    "\n",
    "Noise Levels 10–50: Increasing levels of noise are added to the input data, simulating scenarios with reduced data quality or signal interference.\n",
    "\n",
    "\n",
    "Metrics (MCC, F1, AUC, TPR, TNR, PPV, NPV):\n",
    "\n",
    "MCC (Matthews Correlation Coefficient): Measures the quality of predictions (balanced for imbalanced datasets).\n",
    "\n",
    "F1 Score: Balances precision and recall.\n",
    "\n",
    "AUC (Area Under the Curve): Measures the ability to distinguish between classes.\n",
    "\n",
    "TPR (True Positive Rate): Sensitivity (recall for positives).\n",
    "\n",
    "TNR (True Negative Rate): Specificity (recall for negatives).\n",
    "\n",
    "PPV (Precision for positives): Probability that a positive prediction is correct.\n",
    "\n",
    "NPV (Precision for negatives): Probability that a negative prediction is correct.\n",
    "\n",
    "\n",
    "\"Noise_Level\",\"MCC\",\"F1\",\"AUC\",\"TPR\",\"TNR\",\"PPV\",\"NPV\"\n",
    "\n",
    "0,0.787146199317093,0.978701046312937,0.989827898550725,0.985072463768116,0.75875,0.972690193393951,0.869672438672439\n",
    "\n",
    "10,0.774733683746892,0.977824521118772,0.985471014492754,0.986231884057971,0.7325,0.969842927893636,0.877285714285714\n",
    "\n",
    "20,0.741025278620514,0.976029239802289,0.96786231884058,0.991884057971015,0.64875,0.960935688317965,0.912981240981241\n",
    "\n",
    "30,0.565663317123334,0.962752982763122,0.935969202898551,0.987536231884058,0.4475,0.939441067688016,0.822511904761905\n",
    "\n",
    "40,0.495896101953109,0.94892690181517,0.897083333333333,0.978985507246377,0.35375,0.920831466864912,0.81527380952381\n",
    "\n",
    "50,0.544160688667241,0.933970303115782,0.889184782608696,0.962753623188406,0.39125,0.907020314035807,0.857488095238095\n",
    "\n",
    "\n",
    "**Impact of Noise**:\n",
    "\n",
    "Moderate Noise (10–20): The model shows good robustness, with minor declines in MCC, F1, and AUC.\n",
    "\n",
    "High Noise (30–50): Significant degradation in performance, particularly in MCC, AUC, and TNR. This suggests the model  struggles with highly noisy conditions.\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b79f4d5b",
   "metadata": {},
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.16"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
