{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "1659a8c9",
   "metadata": {},
   "source": [
    "# Dataset: GSE137342\n",
    "\n",
    "This dataset is part of a large-scale clinical transcriptomic analysis focused on sepsis patients in an Indian cohort. The study investigates blood gene expression in patients suffering from severe sepsis and septic shock admitted to the ICU. The goal of this research was to understand the transcriptional changes associated with sepsis and septic shock.\n",
    "\n",
    "By utilizing transcriptional module scores, the study quantifies biological processes related to immune response, inflammation, and sepsis progression. The findings indicate that genes related to immune response were more suppressed, while inflammation-related genes were more active in sepsis patients. These insights could be valuable for developing diagnostic biomarkers and prognostic tools for sepsis management.\n",
    "\n",
    "### Overall Design:\n",
    "\n",
    "Patients and Controls:\n",
    "\n",
    "The study included 27 sepsis patients and 12 healthy controls.The healthy controls were age- and gender-matched but had no inflammatory diseases.\n",
    "\n",
    "Time Points for Sepsis Patients:\n",
    "\n",
    "Day 1 (D1): Blood samples were collected at the time of sepsis diagnosis.\n",
    "\n",
    "Day 2 (D2): A second blood sample was collected 24 hours after diagnosis to observe changes in gene expression over time.\n",
    "\n",
    "Healthy Control Samples:A single blood sample was collected from each healthy individual.\n",
    "\n",
    "The dataset has 2 GPLs which GPL 16686 does not include the healthy control ones, so we just investigated the GPL10558.\n",
    "\n",
    "For sepsis patients they have mixed all sepsis, severe sepsis and septic shock sampling in Day1 and Day2 in the dataset. So we just separate each dataset for each day:\n",
    "\n",
    "Day1 ->\n",
    "\n",
    " 1. sepsis_allD1_GSE137342_10558.csv that includes all sepsis , severe sepsis and septic shock patients with healthy controls.\n",
    " \n",
    "2. sepsis_onlyD1_GSE137342_10558.csv that includes only sepsis and severe sepsis and healthy controls.\n",
    "\n",
    "3. septic_shockD1_GSE137342_10558.csv that includes only septic shock patients and healthy controls.\n",
    "\n",
    "Day 2 ->\n",
    "\n",
    " 1. sepsis_allD2_GSE137342_10558.csv that includes all sepsis , severe sepsis and septic shock patients with healthy controls.(34 rows and 86 columns)\n",
    " \n",
    "2. sepsis_onlyD2_GSE137342_10558.csv that includes only sepsis and severe sepsis and healthy controls. 15 sepsis and 12 healthy samples. (27 rows and 86 columns)\n",
    "\n",
    "3. septic_shockD2_GSE137342_10558.csv that includes only septic shock patients and healthy controls.\n",
    "\n",
    "Present genes: 53\n",
    "\n",
    "Missing genes: 2 (PTX3, MMP8)\n",
    "\n",
    "## sepsis_onlyD2_GSE137342_10558\n",
    "\n",
    "The dataset was already normalized. First the random forest applied to samples from day 2 of only sepsis patients and healthy ones.(sepsis_onlyD2_GSE137342_10558.csv). Because the dataset was almost balanced we didn't use the SMOTE resampling. (Rf-sep-onlyD2-GSE137342-10558code.R)\n",
    "\n",
    "### Random Forest Model:\n",
    "\n",
    "Random forest applied to the dataset 100 times by randomly splitting the dataset to train and test data by 0.8 to predict sepsis patients based on their gene expression data. (repeated_splits_metrics_D2_seponly.csv)\n",
    "\n",
    "### Average Metrics\n",
    "\n",
    "The average of 100 times repeated for each metric (MCC, F1 score, AUC, TPR, TNR,...) is calculated . (average_metrics.csv)\n",
    "\n",
    "\n",
    "\n",
    "MCC     0.924831632475944\n",
    "F1      0.952\n",
    "AUC     1\n",
    "TPR     0.923333333333333\n",
    "TNR     1\n",
    "PPV.    1\n",
    "NPV     0.926666666666667\n",
    "\n",
    "Overall Assessment\n",
    "\n",
    "**MCC** (0.92): 0.9248 indicates strong agreement between predicted and actual labels, meaning the model has high predictive power.\n",
    "\n",
    "**F1 Score** (0.95): 0.952 means the model maintains an excellent balance between precision and recall, meaning it correctly predicts most sepsis cases without many false positives.\n",
    "\n",
    "**AUC** (1): Perfect class separation.\n",
    "\n",
    "**TPR** (0.92) and **TNR** (1): High sensitivity and specificity.\n",
    "\n",
    "**PPV** (1) and **NPV** (0.92): Accurate predictions for both classes. 0.9267 (92.67%) means that when the model predicts a patient is healthy, it is correct 92.67% of the time.\n",
    "\n",
    "\n",
    "\n",
    "### Feature removal results:\n",
    "\n",
    "Each feature has been removed from the dataset and all metrics values are calculated in each iteration to find out the most important genes that impact the MCC value by its removal from the dataset. (feature_removal_results.csv)\n",
    "\n",
    "The plot shows us MCC calculated after removing each feature(gene) from the dataset.\n",
    "(Impact of feature removal on MCC)\n",
    "\n",
    "Genes with High Impact (MCC drops significantly):(**CCL19, IL8 , IL6, TREM1, VCAM1, HLA-DRA**)\n",
    "Removing these genes causes a sharp decline in MCC, indicating they are critical for the model's classification performance.\n",
    "These are likely the most informative genes for predicting the outcome (e.g., sepsis).\n",
    "\n",
    "Genes with Low Impact (Minimal change in MCC):(**MAPK14.3, BCL2.3, CCL25, IL1R2.1, TLR4, CCR2**)\n",
    "Removing these genes has little to no effect on MCC, suggesting they are less important for the model’s classification task.\n",
    "\n",
    "### Sanity Check:\n",
    "\n",
    "This plot represents the sanity check where different metrics are tracked as noise levels increase.(sanity_check_results_D2_seponly.csv)\n",
    "\n",
    "Impact of Noise on Metrics:\n",
    "\n",
    "The model performs well under low noise conditions but degrades significantly after 30% noise. The AUC remains relatively high, meaning the model still ranks sepsis vs. non-sepsis cases well. MCC suffers the most, indicating that noise impacts the overall reliability of predictions.\n",
    "But all in all the dataset is small and this the number of each label in the dataset:\n",
    "table(data$Label)\n",
    "\n",
    "Healthy  Sepsis \n",
    "     12      15 \n",
    "> table(test_data$Label)\n",
    "\n",
    "Healthy  Sepsis \n",
    "      2       3 \n",
    "> table(train_data$Label)\n",
    "\n",
    "Healthy  Sepsis \n",
    "     10      12\n",
    "\n",
    "\n",
    "## sepsis_allD2_GSE137342_10558\n",
    "\n",
    "Then the random forest is again applied on the dataset that we prepared and includes both sepsis and septic shock patients and healthy controls.(sepsis_allD2_GSE137342_10558.csv)\n",
    "(Rf-allD2-GSE137342-10558code.R)\n",
    "\n",
    "### Random Forest Model\n",
    "\n",
    "Random forest applied to the dataset 100 times by randomly splitting the dataset to train and test data by 0.8 to predict sepsis patients based on their gene expression data. (repeated_splits_metrics_allD2.csv)\n",
    "\n",
    "### Average Metrics\n",
    "\n",
    "The average of 100 times repeated for each metric (MCC, F1 score, AUC, TPR, TNR,...) is calculated . (average_metrics_allD2.csv)\n",
    "\n",
    "MCC    0.959852813742386\n",
    "F1     0.979523809523809\n",
    "AUC    1\n",
    "TPR    0.965\n",
    "TNR    1\n",
    "PPV    1\n",
    "NPV    0.955\n",
    "\n",
    "\n",
    "\n",
    "### Feature removal results\n",
    "\n",
    "Each feature has been removed from the dataset and all metrics values are calculated in each iteration to find out the most important genes that impact the MCC value by its removal from the dataset. (feature_removal_results_allD2.csv)\n",
    "\n",
    "Genes with High Impact (MCC drops significantly):(**CCL25.1, CRP , MAPK14.3, CCL2, LCN2, HLA-DRA**)\n",
    "Removing these genes causes a sharp decline in MCC, indicating they are critical for the model's classification performance.\n",
    "These are likely the most informative genes for predicting the outcome (e.g., sepsis).\n",
    "\n",
    "Genes with Low Impact (Minimal change in MCC):(**CX3CR1.1, CCR7, ICAM1, TLR2, IL10.1, FCGR1A**)\n",
    "Removing these genes has little to no effect on MCC, suggesting they are less important for the model’s classification task.\n",
    "\n",
    "### sanity check\n",
    "\n",
    "This plot represents the sanity check where different metrics are tracked as noise levels increase.(sanity_check_results_allD2.csv)\n",
    "\n",
    "Impact of Noise on Metrics: The model performs well under low noise conditions but degrades significantly after 30% noise. The AUC remains relatively high, meaning the model still ranks sepsis vs. non-sepsis cases well. MCC suffers the most, indicating that noise impacts the overall reliability of predictions.\n",
    "\n",
    "Because the dataset was imbalanced in the number of sepsis and healthy ones , we performed a resampling technique to make a balance between our target labels.\n",
    "\n",
    " table(data$Label)\n",
    "Healthy  Sepsis \n",
    "     12      22\n",
    "\n",
    "At first we performed the ROSE resampling which did not result properly. The average metrics became \n",
    "MCC : -0.8929899\n",
    " F1 : 0.3181818\n",
    " AUC : 0.99875\n",
    " TPR : 0.095 \n",
    "TNR : 0\n",
    " PPV : 0.1183333\n",
    " NPV : 0\n",
    " \n",
    " > table(test_data$Label) Healthy Sepsis 2 4 \n",
    "> table(train_data$Label) Sepsis Healthy 14 14 \n",
    "\n",
    "The results may be because ROSE uses a more general approach for generating new samples via a kernel density estimate or random perturbations, so it can potentially create more diverse synthetic examples but also sometimes less “structured” ones, and ROSE may oversample minority instances and undersample majority class in a broader manner, sometimes producing more “random” points, which can be good or bad depending on the structure of your data.\n",
    "\n",
    "So we performed SMOTE resampling because For each minority class example, SMOTE finds its k nearest neighbors (in feature space) and randomly interpolates between them to create synthetic points. SMOTE focuses on minority points and tries to stretch or “fill in” the region around them. This can improve how the model learns the minority class boundary. Thus SMOTE can be a great alternative if we have numeric data and want to create more “realistic” synthetic minority points by interpolation.\n",
    "\n",
    "## sepsis_allD2_GSE137342_10558_SMOTE\n",
    "\n",
    "Then the random forest is again applied on the dataset that we prepared and includes both sepsis and septic shock patients and healthy controls.(sepsis_allD2_GSE137342_10558.csv)\n",
    "(Rf-smote-allD2-GSE137342-10558code.R)\n",
    "\n",
    "### Random Forest Model:\n",
    "Random forest applied to the dataset 100 times by randomly splitting the dataset to train and test data by 0.8 to predict sepsis patients based on their gene expression data. (repeated_splits_metrics_smote_allD2.csv)\n",
    "\n",
    "### Average Metrics\n",
    "\n",
    "The average of 100 times repeated for each metric (MCC, F1 score, AUC, TPR, TNR,...) is calculated . (average_metrics_smote_allD2.csv)\n",
    "\n",
    "MCC    0.931421356237309\n",
    "F1     0.964761904761905\n",
    "AUC    0.99875\n",
    "TPR    0.94\n",
    "TNR    1\n",
    "PPV    1\n",
    "NPV    0.923333333333333\n",
    "\n",
    "\n",
    "\n",
    "### Feature removal results:\n",
    "\n",
    "Each feature has been removed from the dataset and all metrics values are calculated in each iteration to find out the most important genes that impact the MCC value by its removal from the dataset. (feature_removal_results_smote_allD2.csv)\n",
    "\n",
    "Genes with High Impact (MCC drops significantly):(**IFNG, PLAUR.2 , TLR2, C3AR1, TREM1, ELANE**)\n",
    "Removing these genes causes a sharp decline in MCC, indicating they are critical for the model's classification performance.\n",
    "These are likely the most informative genes for predicting the outcome (e.g., sepsis).\n",
    "\n",
    "Genes with Low Impact (Minimal change in MCC):(**BCL2, PLAUR, CCL2, CD14, CD14.1, MAPK14.3**)\n",
    "Removing these genes has little to no effect on MCC, suggesting they are less important for the model’s classification task.\n",
    "\n",
    "### Sanity Check:\n",
    "\n",
    "This plot represents the sanity check where different metrics are tracked as noise levels increase.(sanity_check_results_smote_allD2.csv)\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.16"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
