{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "feb6d28f",
   "metadata": {},
   "source": [
    "Datasets: Expression Omnibus datasets : \n",
    "\n",
    "1.**GSE154918** incorporated 26 sepsis patients alongside 14 controls:Strong differences between patients groups, specific expression changes for healthy control (Hlty), uncomplicated infection (Inf1_P), sepsis (Seps_P), septic shock (Shock_P), follow-up of sepsis (Seps_FU), follow-up of septic shock (Shock_FU) groups.\n",
    "\n",
    "\n",
    "2.**GSE131761** contained blood samples from 81 sepsis patients and 15 healthy controls, \n",
    "\n",
    "3.**GSE4607** ,Systemic inflammatory response syndrome(SIR) and septic shock\n",
    "\n",
    "4.**GSE13904**, Normal children, children with SIRS, children with sepsis, and children with septic shock.\n",
    "\n",
    "5.**GSE26378** Expression data from 82 children with septic shock and 21 normal controls \n",
    "\n",
    "6.**GSE26440** Expression data from 98 children with septic shock and 32 normal controls\n",
    "\n",
    "7.**GSE95233**, 51 septic shock patients and 22 healthy volunteers were included in this study. Septic shock patients were sampled twice, at admission, and a second time at D2 or D3. Admission samples from septic shock patients were compared to healthy volunteers, and according to day 28 survival status. Modulation of gene expression between the 2 time points was also analyzed according to day 28 survival.\n",
    "\n",
    "8.**GSE13205** Skeletal muscle transcriptome in ICU patients suffering from sepsis induced multiple organ failure, 13 septic samples, 8 controls\n",
    "\n",
    "9.**GSE54514** Whole blood samples were collected daily for up to 5 days from patients admitted to the intensive care unit with sepsis. RNA isolated from whole blood samples was assayed on Illumina HT-12 gene expression microarrays consisting of 48,804 probes. Microarray analysis identified 3677 genes as differentially expressed across 5 days between septic patients and healthy controls.Daily PAXgene samples for up to 5 days for sepsis survivors (n=26), sepsis nonsurvivors (n=9), and healthy controls (n=18).\n",
    "\n",
    "\n",
    "10.**GSE5772** Differentially expressed genes are identified from a training set of 44 patients. The predictive accuracy of these genes is then assessed in an independent validation set of 50 patients.\n",
    "Training set: GSM134895 - GSM135279\n",
    "Validation set: GSM135655 - GSM135704\n",
    "\n",
    "11.**GSE6535** There was no difference in the expression profile between gram-positive and gram-negative sepsis. The finding remained unchanged even when genes with lower expression level were included or after statistical stringency was lowered. There were, however, ninety-four genes differentially expressed between sepsis and control patients.compared the gene-expression profiles of gram-positive sepsis and gram-negative sepsis in critically ill patients.\n",
    "\n",
    "12.**GSE65517** Comparative analysis of monocytes from healthy donors, patients with metastatic breast cancer, sepsis or tuberculosis.\n",
    "\n",
    "13.**GSE241238** analyzes and compares the test transcripts of peripheral blood mononuclear cells (PBMCs) isolated from 4 healthy donors and 4 septic patients' whole blood.\n",
    "\n",
    "14.**GSE242059** comprehensive Analysis of ceRNA Network Composed of circRNA, miRNA, and mRNA in septic acute kidney injury patients based on RNA-seq.We collected 10 samples of whole blood of adults, including 5 cases of SAKI and 5 cases of healthy controls.\n",
    "\n",
    "15.**GSE232753** Comparative gene expression profiling analysis of RNA-seq data for patients with sepsis and healthy volunteers.\n",
    "\n",
    "16.**GSE216902** We enrolled 37 elderly patients with sepsis at a tertiary referral hospital in central Taiwan. Blood samples and clinical/laboratory data were collected on day-1 and day-8 for RNA-Seq analysis. We implemented the DVID enrichment analysis and Gene Set Enrichment Analysis to identify the dynamic transcriptome and enriched pathway of day-8/day-1 differentially expressed genes, respectively. Finally, we utilized the MiXCR to analyze the diversity of T cell repertoire.Whole transcriptome analysis of critically ill elderly patients with sepsis.\n",
    "\n",
    "17.**GSE182522** A case-control study. A total of 18 immunocompromised subjects with sepsis and 18 sequential organ failure assessment (SOFA) score-matched immunocompetent control subjects were enrolled.\n",
    "\n",
    "18.**GSE175453** A mixture of whole blood myeloid-enriched and Ficoll-enriched peripheral blood mononuclear cells from four late septic patients (post-sepsis day 14-21, RAP n=1 and CCI n=3) and five healthy subjects underwent Cellular Indexing of Transcriptomes and Epitopes by Sequencing (CITE-seq).\n",
    "\n",
    "19.**GSE174507** This case-control study included 40 sepsis patients and 40 physical examination donors as a control group. RNA-seq technology and bioinformatics analysis were used to screen out the DEMs between sepsis patients and normal people. The expression of these DEMs were subsequently verified by qRT-PCR. In addition, interleukin-6 (IL-6), IL-21, CXCL-8, monocyte chemotactic protein-1 (MCP-1), and molecular determination of the expression levels of TDAG8 and TLR4 mRNA were assessed. Finally, the relationship between miRNA-3663-3p and pro-inflammatory cytokines secretion or TDAG8/TLR4 mRNA expression were evaluated by linear correlation analysis.This dataset includes samples from 12 sepsis patients and 6 control donors.\n",
    "\n",
    "20.**GSE136200** RNA-Seq data of purified CD14+ monocytes, CD4+, CD8+ T cells and CD19+ B cells from six critically ill patients with sepsis due to community-acquired pneumonia and four healthy subjects.\n",
    "\n",
    "21.**GSE63311** For the RNA-Seq study reported here, 73 patients were recruited with deferred consent at the time of first examination in an emergency ward based on the opinion of physicians that there was a potential for the patient's condition to develop into sepsis. These were retrospectively divided into groups based on clinical features and compared to 11 non-urgent surgical controls.\n",
    "\n",
    "22.**GSE236713** Biomarker Validation on the ‘Analysis of geNe Expression and bioMarkers fOr poiNt-of-care dEcision support in Sepsis’ (ANEMONES) study. Blood samples were collected from sepsis and SIRS patients at day 1, day 2 and day 5 of admittance to an intensive care unit (ICU) and on discharge from ICU. Some time-points were not collected due to patient death, patients leaving ICU or events beyond our control. Healthy control blood samples were collected from volunteers once and were recorded as day 1 only.\n",
    "\n",
    "23.**GSE168440** Urine was collected from 41 sepsis patients within 12 hours of sepsis onset. Urine was collected from 39 SIRS patients within 4 hours after the end of surgery.\n",
    "\n",
    "24.**GSE69528** Total RNA from whole blood obtained from patients with sepsis caused by B.pseudomallei (n=29) or other pathogens (n=54) and uninfected controls (28 healthy and 27 subjects with type 2 diabetes mellitus) were collected. In order to validate the published signature, microarray data were generated from these samples. This dataset was also used for an independent selection of signatures for septicemic melioidosis. The same RNA samples were used for validation by a high throughput real-time PCR technique, Fluidigm.\n",
    "\n",
    "25.**GSE134347** Whole-blood transcriptional patterns of protein-coding and non-coding RNA molecules in healthy subjects, critically-ill patients with sepsis or non-infectious conditions were generated by means of the human transcriptome array 2.0 (HTA 2.0) chip. Whole-blood from patients was collected within 24 hours of intensive care unit (ICU) admission. Title legend: healthy, healthy subjects; sepsis, ICU patients with sepsis; noninfectious, ICU patients with non-infectious conditions.\n",
    "\n",
    "26.**GSE134364** Protein-coding and non-coding RNA landscape in critically ill patients with sepsis.\n",
    "      \n",
    "27.**GSE134358** Genome-wide gene expression profiling of whole blood leukocytes in critically ill patients with sepsis or non-infectious disease has been used extensively in search of diagnostic biomarkers, as well as prognostic signatures reflecting diseases severity and outcome.\n",
    "\n",
    "28.**GSE121350** This microarray-based approach unveiled the expression of about 87,912 distinct HERV probesets and identified 764 putative promoter LTRs and 642 putative polyA LTRs in whole blood.102 septic shock patients at D1, D3 and D6 after ICU admission.\n",
    "\n",
    "29.**GSE67652** Transcriptional profiling of granulocytes from 24 subjects was used to investigate their differences regarding sepsis and aging.Two-colour hybridization design. Aliquots of total RNA isolated from adult or elder individuals with sepsis or healthy controls were labeled with Cy3 and co-hybridized with a common reference RNA pool (Universal Human Reference RNA, Agilent, cat #740000) labeled with Cy5 . Biological replicates:6 per group. No technical replicates.\n",
    "\n",
    "30.**GSE57065** Twenty-eight ICU patients were enrolled at the onset of septic shock. Blood samples were collected within 30 minutes, 24 and 48 hours after septic shock and compared to twenty-five healthy volunteers.\n",
    "\n",
    "31.**GSE48080** In this study we investigate whole-genome gene expression profiles of mononuclear cells from survivor (n=5) and non-survivor (n=5) septic patients, as well as from 3 healthy controls. Blood samples were collected at the time of sepsis diagnosis and seven days later, allowing us to evaluate the role of biological processes or genes possibly involved in patient recovery. Aiming to circumvent, at least partially, the heterogeneity of septic patients we included only patients admitted with sepsis caused by community-acquired pneumonia.\n",
    "\n",
    "32.**GSE28750** This was a multi-centre, prospective clinical trial conducted across 4 tertiary critical care settings in Australia. Sepsis patients were recruited if they met the 1992 Consensus Statement criteria and had clinical evidence of systemic infection based on microbiology diagnoses (n=27). Participants in the post-surgical (PS) group were recruited pre-operatively and blood samples collected within 24 hours following surgery (n=38). Healthy controls (HC) included hospital staff with no known concurrent illnesses (n=20). Each participant had minimally 5ml of PAXgene blood collected for leucocyte RNA isolation and gene expression analyses. The GEO data represents the preliminary microarray investigation performed on Human U133 Plus 2.0 GeneChips (Affymetrix), assaying 41 patient samples (Sepsis n=10, Post-Surgical n=11, Control n=20).\n",
    "\n",
    "33.**GSE12624** We used peripheral whole blood obtained with the PaxGene system at the time of admission to the ICU, i.e., 12 hours after trauma, for transcriptome analysis and monitored clinically the sequence of events from the admission to the ICU, to the very onset of sepsis and finally to the full development of multiple organ dysfunction syndrome (MODS). Study protocols and standard operating procedures (SOPs) regarding ethics, patient recruitment, logistics in sample acquisition and storage, microarray experiments, data analysis and data management have been developed and provided for a high feasibility and reproducibility of the investigations.Group S (Sepsis): 34 patients.Group N (Non-sepsis): 36 patients.\n",
    "\n",
    "34.**GSE13015** Whole blood obtained from patients with sepsis caused by B.pseudomallei or other pathogens were collected. mRNA Expression level of each patient was determined and compared to mRNA expression level from uninfected control including patient who had recovered from melioidosis, patient diagnosed type 2 diabetes, and healthy donors using microarray analysis.\n",
    "\n",
    "35.**GSE9960** Gene-expression profiling of peripheral blood mononuclear cells in sepsis.To identify signature genes that help distinguish (1) sepsis from non-infectious causes of systemic inflammatory response syndrome, (2) between Gram-positive and Gram-negative sepsis.A total of 70 critically ill patients were enrolled in a single-centre observational study. Gene-expression profiling was performed using Affymetrix microarray (U133plus2) with 54,675 transcripts. Data was divided into a training set (n=35) and a validation set (n=35). A molecular signature was developed in the training set and was then validated in the validation set.\n",
    "\n",
    "36.**GSE5663** Accuracy of sepsis prediction was obtained using cross-validation of gene expression data from 12 human spleen samples and from 16 mouse spleen samples. For blood studies, classifiers were constructed using data from a training data set of 26 microarrays. The error rate of the classifiers was estimated on seven de-identified microarrays, and then on a subsequent cross-validation for all 33 blood microarrays. Estimates of classification accuracy of sepsis in human spleen were 67.1%; in mouse spleen, 96%; and in mouse blood, 94.4% (all estimates were based on nested cross-validation). Lists of genes with substantial changes in expression between study and control groups were used to identify nine mouse common inflammatory response genes, six of which were mapped into a single pathway using contemporary pathway analysis tools.\n",
    "\n",
    "37.**GSE5271** Time evolution study of sepsis in patients at D1, D7 and D28 post-inclusion. The day of inclusion corresponded to the first blood sampling (15 ml) quoted day 0. The following blood samples were performed at days 1, 7, and 28. Since the pre-sepsis expression of the genes as a control cannot be obtained, the day 0 sample was used patient per patient as a reference value for the next day's samples for microarray study. Patients' evolution was considered.\n",
    "\n",
    "38.**GSE5262** Septic shock patients: outcome at D1 (24 hours after inclusion).Microarray was performed on PBMC with an inflammatory dedicated slide of 340 genes, and target samples labelled with Cy3 for day 0 (reference) and with Cy5 for the other days. Changes in expression levels were evaluated by the ratio Dx/D0. According to outcome and time evolution, a two-dimensional clustering was performed.\n",
    "\n",
    "39.**GSE208581** Risk assessment with gene expression markers in sepsis development. Whole blood RNA sequencing analysis was performed on preoperative samples taken from 267 patients. These comprised patients who developed postoperatively infection leading to sepsis (n=77) or an uncomplicated infection outcome (n=49), non-infectious systemic inflammatory response (n=31), or an uncomplicated postoperative course (n=110).\n",
    "\n",
    "40.**GSE243217** We performed the single RNA-seq analysis using whole blood from 22 sepsis patients and 35 COVID-19 patients admitted to our department, as well as 15 healthy subjects.\n",
    "\n",
    "41.**GSE241238** This study aimed to analyze and compare the test transcripts of peripheral blood mononuclear cells (PBMCs) isolated from 4 healthy donors and 4 septic patients' whole blood.\n",
    "\n",
    "42.**GSE252275** Gene expression analysis in platelets of septic patients. Three group comparison: Control patients, septic patients without thrombocytopenia and septic patients with thrombocytopenia.\n",
    "\n",
    "43.**GSE224095** Single-cell transcriptomic changes in sepsis patients. Here we analyzed the whole blood transcriptome of 5 healthy controls and 5 patients with confirmed sepsis from the Department of Emergency, Division of Surgical Critical care, Tongji Trauma Center, Tongji Hospital, Tongji Medical College. This research aims to understand the expression of immune cells in sepsis patients and to construct an immune cell landscape of sepsis.\n",
    "\n",
    "44.**GSE210797** 6 healthy donors and 6 non-viral sepsis patients. A Role for IFITM3 in Regulating Fibrinogen Endocytosis and Platelet Reactivity in Non-Viral Sepsis.\n",
    "\n",
    "45.**GSE199816** Transcriptional Differences Between COVID-19 and Sepsis. Comparing the mRNA levels of whole blood samples from COVID-19 patients with septic patients clarified a pathology that is specific to COVID-19.\n",
    "\n",
    "46.**GSE211210** A total of 10 patients with acute sepsis and 10 health donors were recruited to take the next-generation RNA-seq analysis and total RNA was purified from blood samples from sepsis patients (n = 5) and healthy control subjects (n = 5).\n",
    "\n",
    "47.**GSE186054** Here, 3 sepsis patients and 3 healthy controls were enrolled for analysis of the immunological characteristics of their neutrophils. We sought to comprehensively characterize the transcriptional changes in the two groups by transcriptome sequencing.\n",
    "\n",
    "48.**GSE174183** Patients admitted to the ICU from April 2014 to December 2017 with clinical diagnosis of septic shock and APACHE II score >=8. Patients are sub-categorized into three groups depending on the intact LAP activities. RNA were isolated from blood samples for RNAseq profile.\n",
    "\n",
    "49.**GSE185263** Whole blood RNA-seq and clinical data was collected from 348 patients from four emergency rooms and one intensive care unit.Predicting sepsis severity at first clinical presentation: the role of endotypes and mechanistic signatures.\n",
    "\n",
    "50.**GSE180387** RNAseq of human polymorphonuclear cells isolated from the peripheral blood of patients affected by septic shock.\n",
    "\n",
    "51.**GSE69063** The study design consisted of four disease phenotypes (anaphylaxis, sepsis, trauma, healthy controls) and up to 3 time points. Healthy controls were matched to patients within each study.Gene expression patterns in blood during human anaphylaxis, trauma, and sepsis.\n",
    "\n",
    "52.**GSE168443** Genomic biomarkers from urine cells indicate a unique metabolomic difference between sepsis and sterile inflammation.\n",
    "\n",
    "53.**GSE167363** Single-cell RNA-sequencing of human peripheral blood mononuclear cells from healthy controls, survivor and non-survivor of gram-negative sepsis patients.Dynamic changes in human single cell transcriptional signatures during fatal sepsis.\n",
    "\n",
    "54.**GSE100159** A Transcriptome Fingerprinting Assay for Clinical Immune Monitoring (Sepsis Datasets).Sepsis (35 condition, 12 control, 47 total).\n",
    "\n",
    "55.**GSE121349** Modulation of LTR-retrotransposons expression in mHLA-DR stratified septic shock patients: a pilot study. 10 septic shock patients at D1 and D3-4 after ICU admission presenting with features of sepsis-induced immunosuppression or not (HLA-DR low/high), and compared to 4 healthy controls.\n",
    "\n",
    "56.**GSE121350** Modulation of LTR-retrotransposons expression in septic shock patients: a pilot study.102 septic shock patients at D1, D3 and D6 after ICU admission.\n",
    "\n",
    "\n",
    "\t\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.16"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
