{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "8a453d98",
   "metadata": {},
   "source": [
    "# Sepsis Diagnostic genes\n",
    "\n",
    "\n",
    "## neonatal sepsis: \n",
    "\n",
    "**GSE25504** , Whole blood mRNA expression profiling of host molecular networks in neonatal sepsis.\n",
    "GLP6947; control + infected, peripheral boold.\n",
    "\n",
    "**EMTAB4785**: normal, neonatal sepsis, blood.\n",
    "\n",
    "\n",
    "## Pediatric sepsis:\n",
    "\n",
    "**GSE66099** Unique Patients from the Genomics of Pediatric SIRS and Septic Shock Investigators (GPSSSI)\n",
    "Normal children + septic shock children. Whole blood.\n",
    "\n",
    "**GSE8121** Pediatric septic shock.\n",
    " Normal children + septic shock children. Whole blood.\n",
    "\n",
    "**GSE9692** Validation of Genome-wide Expression patterns in Pediatric Septic Shock.\n",
    "Normal children + septic shock children. Whole blood.\n",
    "\n",
    "**GSE13904**,Normal children, children with SIRS, children with sepsis, and children with septic shock.\n",
    "Normal children + children with SIRs + with sepsis + with septic shock. Whole blood.\n",
    "\n",
    "**GSE26378** Expression data from 82 children with septic shock and 21 normal controls.\n",
    "Children with septic shock + control (survivor, non survivor). Whole blood.\n",
    " \n",
    "**GSE26440** Expression data from 98 children with septic shock and 32 normal controls.\n",
    "Children with septic shock + control (survivor, non survivor). Whole blood.\n",
    "\n",
    "**GSE72829** (GLP6883) Diagnosis of childhood bacterial and viral infection using host RNA expression\n",
    "Children with sepsis + maningitis sepsis + control . whole blood.\n",
    "\n",
    "**GSE4607**,Systemic inflammatory response syndrome(SIR) and septic shock .\n",
    "Children with Septic shock + SIRs + control. (survivor, non survivor). Whole blood.\n",
    "\n",
    "\n",
    "## Septic shock:\n",
    "\n",
    "\n",
    "**GSE95233** 51 septic shock patients and 22 healthy volunteers were included in this study. Septic shock patients were sampled twice, at admission, and a second time at D2 or D3. Admission samples from septic shock patients were compared to healthy volunteers, and according to day 28 survival status. Modulation of gene expression between the 2 time points was also analyzed according to day 28 survival.\n",
    "Patient in D1/2/3 + control . (survivor, non survivor).\n",
    "\n",
    "**GSE121350** This microarray-based approach unveiled the expression of about 87,912 distinct HERV probesets and identified 764 putative promoter LTRs and 642 putative polyA LTRs in whole blood.102 septic shock patients at D1, D3 and D6 after ICU admission.\n",
    "Septic shock + healthy control. D1/3/6. Whole blood.\n",
    "\n",
    "**GSE57065** Twenty-eight ICU patients were enrolled at the onset of septic shock. Blood samples were collected within 30 minutes, 24 and 48 hours after septic shock and compared to twenty-five healthy volunteers.\n",
    "septic shock + healthy volunteers. h0/24/48. Blood leukocytes.\n",
    "\n",
    "**GSE5262** Septic shock patients: outcome at D1 (24 hours after inclusion).Microarray was performed on PBMC with an inflammatory dedicated slide of 340 genes, and target samples labelled with Cy3 for day 0 (reference) and with Cy5 for the other days. Changes in expression levels were evaluated by the ratio Dx/D0. According to outcome and time evolution, a two-dimensional clustering was performed. XXX\n",
    "\n",
    "**GSE174183** Patients admitted to the ICU from April 2014 to December 2017 with clinical diagnosis of septic shock and APACHE II score >=8. Patients are sub-categorized into three groups depending on the intact LAP activities. RNA were isolated from blood samples for RNAseq profile.\n",
    "Septic shock patient with 3 level depending on the intact of LC3-associated phagocytosis(LAP) activities.Blood samples.\n",
    "\n",
    "**GSE180387** RNAseq of human polymorphonuclear cells isolated from the peripheral blood of patients affected by septic shock.\n",
    "Septic shock paients.(survivor, non survivor). Polymorphonuclear cell from peripheral blood.\n",
    "\n",
    "**GSE121349** Modulation of LTR-retrotransposons expression in mHLA-DR stratified septic shock patients: a pilot study. 10 septic shock patients at D1 and D3-4 after ICU admission presenting with features of sepsis-induced immunosuppression or not (HLA-DR low/high), and compared to 4 healthy controls.\n",
    "Septic shock + healyhy. Whole blood.\n",
    "\n",
    "**GSE110487** We analyzed 31 patients (17 Responders and 14 Not Responders to early therapy). For each patient, 2 samples were collected. In particular the first sample (T1) collected within 16 hours from ICU admission whereas the second (T2) collected within 48 hours from ICU admission. Experimental groups (Responders and Nonresponders) are defined accordingly with SOFA scores improvements within 48 hours.Whole blood transcriptome analysis of Septic shock patients according to early therapy response.\n",
    "Patient after septic shock+ after early therapy response. Whole blood.\n",
    "\n",
    "**GSE33119** In total 20 patients were tested before and after specific treatment.Transcriptome of the septic shock after treatment by activated protein C.\n",
    "Septic shock + after treatment. (survivor, non survivor). Whole blood.\n",
    "\n",
    "**GSE64457**, 9 septic shock patients at D3-4 and D6-8 presenting with features of sepsis-induced immunosuppression and compared to 8 healthy controls.Marked alterations of neutrophil functions during sepsis-induced immunosuppression.\n",
    "Septic shock + healthy control. Purified neutrophil.\n",
    "\n",
    "**E-MTAB-1548** The peripheral whole-blood transcriptome analysis has been used to the identification of biomarkers in different clinical syndromes. The ability to simultaneously measure the gene expression levels of the whole transcriptome is providing a broader view of clinical syndromes such as sepsis. The aim of this study was to assess the gene expression profiles of post-surgical patients with septic shock compared with patients without sepsis.\n",
    "SIRs + septic shock + normal.  (survivor, non survivor). peripheral blood\n",
    "\n",
    "**GSE33118** Septic shock by pneumopathy was studied in prospective way in 20 patients and compared to normal blood. Blood samples were taken within 12 hours of diagnosis and the resulting transcritomes were compared to 42 normal control samples (selected samples from series GSE10715, GSE12711, GSE16728 and GSE7400).Transcriptome of the septic shock.\n",
    "Septic shock by pneumopathy + normal . blood samples.\n",
    "\n",
    "**GSE123729** AKIRIN1 is a suitable reference gene in natural killer cells and granulocytes from patients with SIRS and septic shock. We enrolled critically ill patients and collected blood on admission to our surgical intensive care unit if, within the preceding 24 hours, they had a diagnosis of post-traumatic systemic inflammatory response syndrome (SIRS) or septic shock. Hospitalized noncritically ill control patients were recruited on presurgical examination. To transcriptionally characterize the innate immune response to sterile (SIRS) and infectious (septic shock) systemic inflammation, granulocytes were isolated from patient blood for RNA extraction and hybridization on Affymetrix microarrays.\n",
    "Septic shock + SIRs + presurgical patients. Peripheral granulocytes.\n",
    "\n",
    "**E-MTAB-7581** The aim of the experiment was to assign patients enrolled in the VANISH randomised trial to sepsis response signature (SRS) endotypes based on a previously published gene expression signature, in order to test for differential responses to treatment. VANISH was a double-blind randomised clinical trial in septic shock, with patients randomised to receive norepinephrine or vasopressin followed by hydrocortisone or placebo. We collected blood samples upon enrolment, extracted RNA and performed transcriptomic profiling using microarrays, allocated patients to SRS1 or SRS2 using a linear model (Davenport 2016), and tested for an association between sepsis endotype and response to either norepinephrine or vasopressin, or to corticosteroids. There was a significant interaction between treatment with hydrocortisone or placebo, and SRS endotype (p=0Â·02).\n",
    "septic shock patients + after treatment. (survivor, non survivor). Blood sapmles.\n",
    "\n",
    "\n",
    "**GSE106878** Paired samples of 47 septic shock patients from the CORTICUS-trial (ClinicalTrials.gov number, NCT00147004) treated with either Placebo or Hydrocortisone (HC) were included. Whole blood from patients was collected in the morning (pre) and 24 hours (post) later. Total cellular RNA from circulating leukocytes was isolated using PaxGene Blood RNA Kit (PreAnalytix) for transcriptome profiling.\n",
    "Septic shock patients + after treatment. (survivor, non survivor). Blood leukocytes.\n",
    "\n",
    "**GSE48080** In this study we investigate whole-genome gene expression profiles of mononuclear cells from survivor (n=5) and non-survivor (n=5) septic patients, as well as from 3 healthy controls. Blood samples were collected at the time of sepsis diagnosis and seven days later, allowing us to evaluate the role of biological processes or genes possibly involved in patient recovery. Aiming to circumvent, at least partially, the heterogeneity of septic patients we included only patients admitted with sepsis caused by community-acquired pneumonia.\n",
    "Septic shock + severe spsis + healthy control. (survivor, non survivor). D0/7. mononculear cells.\n",
    "\n",
    "**GSE131761** contained blood samples from 81 sepsis patients and 15 healthy controls.\n",
    "Post surgical septic shock + non septic shock + control patient. Peripheral blood. \n",
    "\n",
    "\n",
    "## Sepsis, Peripheral Blood samples:\n",
    "\n",
    "**GSE69063** The study design consisted of four disease phenotypes (anaphylaxis, sepsis, trauma, healthy controls) and up to 3 time points. Healthy controls were matched to patients within each study.Gene expression patterns in blood during human anaphylaxis, trauma, and sepsis.\n",
    "Sepsis patients + anaphylaxis + trauma + healthy control. \n",
    "\n",
    "**GSE12624** We used peripheral whole blood obtained with the PaxGene system at the time of admission to the ICU, i.e., 12 hours after trauma, for transcriptome analysis and monitored clinically the sequence of events from the admission to the ICU, to the very onset of sepsis and finally to the full development of multiple organ dysfunction syndrome (MODS). Study protocols and standard operating procedures (SOPs) regarding ethics, patient recruitment, logistics in sample acquisition and storage, microarray experiments, data analysis and data management have been developed and provided for a high feasibility and reproducibility of the investigations.Group S (Sepsis): 34 patients.Group N (Non-sepsis): 36 patients.\n",
    "Polytrauma patients with sepsis + non sepsis. \n",
    "\n",
    "**GSE154918** incorporated 26 sepsis patients alongside 14 controls:Strong differences between patients groups, specific expression changes for healthy control (Hlty), uncomplicated infection (Inf1_P), sepsis (Seps_P), septic shock (Shock_P), follow-up of sepsis (Seps_FU), follow-up of septic shock (Shock_FU) groups.\n",
    "sepsis + septic shock + follow-up of sepsis + follow-up of septic shock+infection+ control.\n",
    "\n",
    "\n",
    "\n",
    "## Sepsis, Whole Blood Datasets:\n",
    "\n",
    "\n",
    "**GSE232753** Comparative gene expression profiling analysis of RNA-seq data for patients with sepsis and healthy volunteers. 20 sepsis , 8 controls.\n",
    "Sepsis + control . D0.\n",
    "\n",
    "**GSE63311** For the RNA-Seq study reported here, 73 patients were recruited with deferred consent at the time of first examination in an emergency ward based on the opinion of physicians that there was a potential for the patient's condition to develop into sepsis. These were retrospectively divided into groups based on clinical features and compared to 11 non-urgent surgical controls.\n",
    "Sepsis + no sepsis surgical control+ control. \n",
    "\n",
    "**GSE28750** This was a multi-centre, prospective clinical trial conducted across 4 tertiary critical care settings in Australia. Sepsis patients were recruited if they met the 1992 Consensus Statement criteria and had clinical evidence of systemic infection based on microbiology diagnoses (n=27). Participants in the post-surgical (PS) group were recruited pre-operatively and blood samples collected within 24 hours following surgery (n=38). Healthy controls (HC) included hospital staff with no known concurrent illnesses (n=20). Each participant had minimally 5ml of PAXgene blood collected for leucocyte RNA isolation and gene expression analyses. The GEO data represents the preliminary microarray investigation performed on Human U133 Plus 2.0 GeneChips (Affymetrix), assaying 41 patient samples (Sepsis n=10, Post-Surgical n=11, Control n=20).\n",
    "Sepsis + post surgical + control. \n",
    "\n",
    "**GSE54514** Whole blood samples were collected daily for up to 5 days from patients admitted to the intensive care unit with sepsis. RNA isolated from whole blood samples was assayed on Illumina HT-12 gene expression microarrays consisting of 48,804 probes. Microarray analysis identified 3677 genes as differentially expressed across 5 days between septic patients and healthy controls.Daily PAXgene samples for up to 5 days for sepsis survivors (n=26), sepsis nonsurvivors (n=9), and healthy controls (n=18).\n",
    "Sepsis + control. (survivor, non survivor). D1/2/3/4/5\n",
    "\n",
    "**GSE242059** comprehensive Analysis of ceRNA Network Composed of circRNA, miRNA, and mRNA in septic acute kidney injury patients based on RNA-seq.We collected 10 samples of whole blood of adults, including 5 cases of SAKI and 5 cases of healthy controls.\n",
    "Septis acute kidney injury + healthy. \n",
    "\n",
    "**GSE69528** Total RNA from whole blood obtained from patients with sepsis caused by B.pseudomallei (n=29) or other pathogens (n=54) and uninfected controls (28 healthy and 27 subjects with type 2 diabetes mellitus) were collected. In order to validate the published signature, microarray data were generated from these samples. This dataset was also used for an independent selection of signatures for septicemic melioidosis. The same RNA samples were used for validation by a high throughput real-time PCR technique, Fluidigm.\n",
    "Sepsis caused by different pathogens + healthy control. \n",
    "\n",
    "**GSE13015** Whole blood obtained from patients with sepsis caused by B.pseudomallei or other pathogens were collected. mRNA Expression level of each patient was determined and compared to mRNA expression level from uninfected control including patient who had recovered from melioidosis, patient diagnosed type 2 diabetes, and healthy donors using microarray analysis.\n",
    "Sepsis caused by diffrent pathogens + healthy controls. \n",
    "\n",
    "**GSE208581** Risk assessment with gene expression markers in sepsis development. Whole blood RNA sequencing analysis was performed on preoperative samples taken from 267 patients. These comprised patients who developed postoperatively infection leading to sepsis (n=77) or an uncomplicated infection outcome (n=49), non-infectious systemic inflammatory response (n=31), or an uncomplicated postoperative course (n=110).\n",
    "preoperative samples f sepsis + infection + â¦ . \n",
    "\n",
    "**GSE224095** ,Single-cell transcriptomic changes in sepsis patients. Here we analyzed the whole blood transcriptome of 5 healthy controls and 5 patients with confirmed sepsis from the Department of Emergency, Division of Surgical Critical care, Tongji Trauma Center, Tongji Hospital, Tongji Medical College. This research aims to understand the expression of immune cells in sepsis patients and to construct an immune cell landscape of sepsis.\n",
    "Sepsis + control.\n",
    "\n",
    "**GSE185263** Whole blood RNA-seq and clinical data was collected from 348 patients from four emergency rooms and one intensive care unit.Predicting sepsis severity at first clinical presentation: the role of endotypes and mechanistic signatures.\n",
    "Sepsis + healthy. (survivor, non survivor). Sofa score.\n",
    "\n",
    "**GSE32707**, A microarray analysis involving whole blood samples isolated from critically ill patients in the medical intensive care unit at Brigham and Women's Hospital. Four groups of intubated subjects undergoing mechanical ventilation were recruited for the study: those with sepsis alone (Sepsis), those with sepsis + ARDS (se/ARDS), those with SIRS (SIRS), and those without sepsis, SIRS, or ARDS (untreated). Blood was obtained from patients on the day of admission (day 0) and 7 days later. RNA was isolated from the whole blood samples and microarrays were prepared to determine differential gene expression between the four groups.\n",
    "Sepsis + sepsis/ARDS + SIRS + patient with non of them. D0/7.\n",
    "\n",
    "**GSE211210** A total of 10 patients with acute sepsis and 10 health donors were recruited to take the next-generation RNA-seq analysis and total RNA was purified from blood samples from sepsis patients (n = 5) and healthy control subjects (n = 5).\n",
    "Acute sepsis + healthy controls.\n",
    "\n",
    "**GSE100159** A Transcriptome Fingerprinting Assay for Clinical Immune Monitoring (Sepsis Datasets).Sepsis (35 condition, 12 control, 47 total).\n",
    "Sepsis caused infections + controls.\n",
    "\n",
    "**GSE60424** We performed whole genome RNAseq profiling of immune cell subsets and whole blood from subjects with an array of immune-associated diseases.This study compared whole transcriptome signatures of 6 immune cell subsets and whole blood from patients with an array of immune-associated diseases. Fresh blood samples were collected from healthy subjects and subjects diagnosed type 1 diabetes, amyotrophic lateral sclerosis, and sepsis, as well as multiple sclerosis patients before and 24 hours after the first treatment with IFN-beta.\n",
    "Sepsis + diabetes1 + ALS + healthy . whole blood and other (B cell, monocytes, â¦)\n",
    "\n",
    "**GSE29536** Whole Blood Transcriptional Modules generated on Illumina Hu-6 V2 Beadchips.This series combines nine independent datasets representing a spectrum of human pathologies expected to result in changes in gene abundance related to changes in expression or cellular composition of whole blood. These nine datasets are composed of 410 individual whole blood profiles generated from patients with HIV, tuberculosis, sepsis, systemic lupus erythematosus, systemic arthritis, B-cell deficiency and liver transplant. For each dataset healthy controls are also included.\n",
    "Sepsis + HIV + tuberculosis + .. .+ healthy control.\n",
    "\n",
    "\n",
    "## peripheral blood mononuclear cells (PBMCs):\n",
    "\n",
    "**GSE241238**  analyzes and compares the test transcripts of peripheral blood mononuclear cells (PBMCs) isolated from 4 healthy donors and 4 septic patients' whole blood.\n",
    "Sepsis + helthy control. \n",
    "\n",
    "**GSE5271** Time evolution study of sepsis in patients at D1, D7 and D28 post-inclusion. The day of inclusion corresponded to the first blood sampling (15 ml) quoted day 0. The following blood samples were performed at days 1, 7, and 28. Since the pre-sepsis expression of the genes as a control cannot be obtained, the day 0 sample was used patient per patient as a reference value for the next day's samples for microarray study. Patients' evolution was considered.\n",
    "Sepsis patients. D1/7/28. Focused only on 340 inflammatory genes.\n",
    "\n",
    "**GSE199816** Transcriptional Differences Between COVID-19 and Sepsis. Comparing the mRNA levels of whole blood samples from COVID-19 patients with septic patients clarified a pathology that is specific to COVID-19.\n",
    "Sepsis + COVID patients.\n",
    "\n",
    "**GSE216902** We enrolled 37 elderly patients with sepsis at a tertiary referral hospital in central Taiwan. Blood samples and clinical/laboratory data were collected on day-1 and day-8 for RNA-Seq analysis. We implemented the DVID enrichment analysis and Gene Set Enrichment Analysis to identify the dynamic transcriptome and enriched pathway of day-8/day-1 differentially expressed genes, respectively. Finally, we utilized the MiXCR to analyze the diversity of T cell repertoire.Whole transcriptome analysis of critically ill elderly patients with sepsis.\n",
    "Sepsis patients. Responder and non responder D1/8. \n",
    "\n",
    "**GSE243217** We performed the single RNA-seq analysis using whole blood from 22 sepsis patients and 35 COVID-19 patients admitted to our department, as well as 15 healthy subjects.\n",
    "Sepsis + COVID + healthy . \n",
    "\n",
    "**GSE109704** The study population consisted of 30 melioidosis cases, 10 healthy controls and 10 sepsis cases caused by other pathogens. Total RNA was extracted from peripheral blood mononuclear cells (PBMCâs) of study subjects. Gene expression profiles of 25 gene targets including 19 immune response genes and 6 epigenetic factors were analyzed by real time quantitative polymerase chain reaction (RT-qPCR).\n",
    "Sepsis patients + melioidosis + helathy control. 25 preselected genes.\n",
    "\n",
    "**GSE49754** ,PBMCs were isolated a healthy donor. Plasma samples were obtained from a first set of patients with culture-confirmed sepsis (n=29) and from uninfected controls (n=17). PBMCs were cultured for 6 h in medium alone, plasma from patients with sepsis, plasma from uninfected controls, and LPS using a final concentration of 20%. Transcriptional profiles were acquired using Illumina HumanHT12 V4 BeadChips.\n",
    "Medium alone (control).\n",
    "Plasma from sepsis patients.\n",
    "Plasma from uninfected controls.\n",
    "Lipopolysaccharide (LPS) as a positive control for immune activation.\n",
    "\n",
    "**GSE175453** A mixture of whole blood myeloid-enriched and Ficoll-enriched peripheral blood mononuclear cells from four late septic patients (post-sepsis day 14-21, RAP n=1 and CCI n=3) and five healthy subjects underwent Cellular Indexing of Transcriptomes and Epitopes by Sequencing (CITE-seq).\n",
    "unique transcriptomic patterns in non-myeloid immune cells during late sepsis.D14-21.\n",
    "Septic patients(single-cell RNA sequencing).\n",
    " \n",
    "**SCP548** dataset focused on sepsis. It comprises single-cell RNA sequencing (scRNA-seq) data from peripheral blood mononuclear cells (PBMCs) of individuals diagnosed with bacterial sepsis. The dataset includes samples from 29 sepsis patients and 36 control subjects across three clinical cohorts.Single Cell Portal\n",
    "Sepsis + control. Single cell RNA-seq.\n",
    "\n",
    "**GSE182522** A case-control study. A total of 18 immunocompromised subjects with sepsis and 18 sequential organ failure assessment (SOFA) score-matched immunocompetent control subjects were enrolled.\n",
    "Sepsis + SOFA + control. \n",
    "\n",
    "**GSE48080** In this study we investigate whole-genome gene expression profiles of mononuclear cells from survivor (n=5) and non-survivor (n=5) septic patients, as well as from 3 healthy controls. Blood samples were collected at the time of sepsis diagnosis and seven days later, allowing us to evaluate the role of biological processes or genes possibly involved in patient recovery. Aiming to circumvent, at least partially, the heterogeneity of septic patients we included only patients admitted with sepsis caused by community-acquired pneumonia.\n",
    "Severe sepsis +septic shock + healthy control .(survivor, non survivor).\n",
    "\n",
    "**GSE9960** Gene-expression profiling of peripheral blood mononuclear cells in sepsis.To identify signature genes that help distinguish (1) sepsis from non-infectious causes of systemic inflammatory response syndrome, (2) between Gram-positive and Gram-negative sepsis.A total of 70 critically ill patients were enrolled in a single-centre observational study. Gene-expression profiling was performed using Affymetrix microarray (U133plus2) with 54,675 transcripts. Data was divided into a training set (n=35) and a validation set (n=35). A molecular signature was developed in the training set and was then validated in the validation set.\n",
    "Sepsis caused by different pathogen(g+/-) + healthy controls.\n",
    "\n",
    "**GSE167363** Single-cell RNA-sequencing of human peripheral blood mononuclear cells from healthy controls, survivor and non-survivor of gram-negative sepsis patients.Dynamic changes in human single cell transcriptional signatures during fatal sepsis.\n",
    "G- sepsis patients + healthy control.(survivor, non survivor).\n",
    "\n",
    "**GSE167914** Impaired peripheral mononuclear cell metabolism in patients at risk of developing sepsis.Mitochondrial respiration and gene expression related to mitochondrial function were measured from the peripheral blood of infection and sepsis patients as well as healthy controls.gene expression related to mitochondrial function in PBMCs.\n",
    "Sepsis + control . (survivor, non survivor).\n",
    "\n",
    "\n",
    "## BLood  leukocytes:\n",
    "\n",
    "**GSE134347** Whole-blood transcriptional patterns of protein-coding and non-coding RNA molecules in healthy subjects, critically-ill patients with sepsis or non-infectious conditions were generated by means of the human transcriptome array 2.0 (HTA 2.0) chip. Whole-blood from patients was collected within 24 hours of intensive care unit (ICU) admission. Title legend: healthy, healthy subjects; sepsis, ICU patients with sepsis; noninfectious, ICU patients with non-infectious conditions. \n",
    "Sepsis + non infectious + healthy.\n",
    "\n",
    "**GSE236713** Biomarker Validation on the âAnalysis of geNe Expression and bioMarkers fOr poiNt-of-care dEcision support in Sepsisâ (ANEMONES) study. Blood samples were collected from sepsis and SIRS patients at day 1, day 2 and day 5 of admittance to an intensive care unit (ICU) and on discharge from ICU. Some time-points were not collected due to patient death, patients leaving ICU or events beyond our control. Healthy control blood samples were collected from volunteers once and were recorded as day 1 only.\n",
    "Sepsis + SIRs + controls. (survivor, non survivor). D1/2/5. \n",
    "\n",
    "**GSE134358** Genome-wide gene expression profiling of whole blood leukocytes in critically ill patients with sepsis or non-infectious disease has been used extensively in search of diagnostic biomarkers, as well as prognostic signatures reflecting diseases severity and outcome.\n",
    "Sepsis + non infectious + healthy.\n",
    "\n",
    "**GSE5772** To examine whether gene-expression profiling of circulating leukocytes is different between patients with sepsis as compared to controls. The signature genes identified will then be used to develop a prediction model to assist the diagnosis of sepsis.\n",
    "Sepsis patients + control patients. \n",
    "\n",
    "**GSE65682** The host response in critically ill patients with sepsis, septic shock remains poorly defined. Considerable research has been conducted to accurately distinguish patients with sepsis from those with non-infectious causes of disease. Technological innovations have positioned systems biology at the forefront of biomarker discovery. Analysis of the whole-blood leukocyte transcriptome enables the assessment of thousands of molecular signals beyond simply measuring several proteins in plasma, which for use as biomarkers is important since combinations of biomarkers likely provide more diagnostic accuracy than the measurement of single ones or a few.\n",
    "????????????????????????\n",
    "\n",
    "**GSE134364** Protein-coding and non-coding RNA landscape in critically ill patients with sepsis.\n",
    "Sepsis + noninfectious + patients+ healthy. \n",
    "\n",
    "**E-MTAB-5273**, **E-MTAB-5274** We assayed leukocyte global gene expression for a prospective validation cohort of 221 adult patients admitted to UK intensive care units with sepsis due to community acquired pneumonia or faecal peritonitis. 10 samples from patients scheduled for elective cardiac surgery were also assayed as non-septic controls. We assigned all samples to sepsis response signature groups after performing unsupervised analysis of the transcriptomic data.\n",
    "Sepsis caused by Community-acquired pneumonia and Faecal peritonitis + non septic controls.\n",
    " (survivor, non survivor).\n",
    "\n",
    "**E-MTAB-4451**, **E-MTAB-4421** We assayed leukocyte global gene expression for a prospective validation cohort of 106 adult patients admitted to UK intensive care units with severe sepsis due to community acquired pneumonia.\n",
    "Sepsis CAP . (survivor, non survivor).\n",
    "\n",
    "\n",
    "## Blood neutrophil :\n",
    "\n",
    "**GSE49756, GSE49755, GSE49757** Neutrophils as Reporter Cells: The study used neutrophils, a type of white blood cell important in the immune response, as \"sensors\" to detect immune signals in plasma (the liquid portion of blood containing proteins, antibodies, and signaling molecules).\n",
    "\n",
    "Plasma Incubation:\n",
    "Plasma from different groups (e.g., sepsis patients, healthy controls) was collected and added to neutrophils isolated from a healthy donor.\n",
    "The neutrophils were cultured in vitro (in a controlled laboratory setting) with this plasma for 6 hours to allow interaction between plasma components and the neutrophils.\n",
    "During this time, the plasma molecules (e.g., cytokines, immune proteins) triggered a transcriptional response (changes in gene expression) in the neutrophils.\n",
    "\n",
    "The plasma samples were incubated with neutrophils for 6 hours, along with other conditions:\n",
    "\n",
    "Medium alone (control).\n",
    "\n",
    "Plasma from sepsis patients.\n",
    "\n",
    "Plasma from uninfected controls.\n",
    "\n",
    "Lipopolysaccharide (LPS) as a positive control for immune activation.\n",
    "\n",
    "**GSE6535** Whole blood was taken from each patient on admission to ICU. Neutrophils were separated from whole blood using density-gradient separation with Ficoll-PaqueP P(Amersham).\tThe study included seventy-two critically ill patients admitted to the intensive care unit (ICU) of Nepean Hospital, Sydney, Australia. Of these, fifty-five patients were diagnosed to have sepsis, as confirmed by microbiological culture. The remaining seventeen patients did not have sepsis and were therefore used as controls. Critically ill patients with either gram-positive sepsis or gram-negative sepsis.\n",
    "G+/ G- sepsis patients + control patients.\n",
    "\n",
    "**GSE186054** Here, 3 sepsis patients and 3 healthy controls were enrolled for analysis of the immunological characteristics of their neutrophils. We sought to comprehensively characterize the transcriptional changes in the two groups by transcriptome sequencing.\n",
    "Sepsis + healthy. \n",
    "\n",
    "**GSE67652** Transcriptional profiling of granulocytes from 24 subjects was used to investigate their differences regarding sepsis and aging.Two-colour hybridization design. Aliquots of total RNA isolated from adult or elder individuals with sepsis or healthy controls were labeled with Cy3 and co-hybridized with a common reference RNA pool (Universal Human Reference RNA, Agilent, cat #740000) labeled with Cy5 . Biological replicates:6 per group. No technical replicates.\n",
    "Adults and elder with sepsis + adult and elder healthy control.\n",
    "\n",
    "**GSE66890** Clinical study of critically ill patients with sepsis and sepsis-related ARDS with whole blood RNA collected within the first 24 hours of admission Goal of the study was to determine whether biologically relevant genes were identified to be differentially expressed genes in patients with sepsis alone and sepsis with ARDS Prospective observational study, case cohort design.\n",
    " sepsis + sepsis-related ARDS(Acute Respiratory Distress Syndrome).\n",
    " \n",
    " \n",
    "## Blood Monocytes:\n",
    "\n",
    "**GSE49753** monocyte-derived dendritic cells. Ex-vivo experiments.\n",
    "Medium alone (control).\n",
    "Plasma from sepsis patients.\n",
    "Plasma from uninfected controls.\n",
    "Lipopolysaccharide (LPS) as a positive control for immune activation.\n",
    "\n",
    "\n",
    "**GSE65517** Comparative analysis of monocytes from healthy donors, patients with metastatic breast cancer, sepsis or tuberculosis.\n",
    "Sepsis G- + breast cancer + tuberculosis + healthy control.\n",
    "\n",
    "**GSE136200** RNA-Seq data of purified CD14+ monocytes, CD4+, CD8+ T cells and CD19+ B cells from six critically ill patients with sepsis due to community-acquired pneumonia and four healthy subjects.\n",
    "Sepsis + healthy.\n",
    "\n",
    "**GSE139913** Analysis of the RNA expression of circulating CD14+ monocytes of healthy volunteers and intensive care unit (ICU) patients with and without sepsis.Differential gene expression in circulating CD14+ monocytes indicates the prognosis of critically ill patients with sepsis.\n",
    "Sepsis + non sepsis + healthy.\n",
    "\n",
    "**GSE133822**, mRNA profiles of CD4, CD8 and CD14 cells were generated for sepsis, critically-ill non-septic, cancer patients and age-matched healthy controls using Illumina Hiseq 2500.T cell and Monocyte Specific RNAseq Analysis in Septic and Non Septic Critically-Ill Patients and in Patients with Cancer.\n",
    "Sepsis + non sepsis + cancer + healthy.\n",
    "\n",
    "**GSE46955** Transcriptome analysis of blood monocytes from sepsis patients. Blood monocytes from gram-negative sepsis patients during sepsis (Sepsis) and following their recovery (Recovery/Basal) as well as healthy donor (control) were isolated. Thereafter, these cells were treated ex vivo with or without LPS for 3h and analysed for transcriptomic study. \n",
    "Sepsis + recovery from sepsis + healthy.\n",
    "\n",
    "\n",
    "## Urine samples:(4 datasets)\n",
    "\n",
    "**GSE168440** ,Urine was collected from 41 sepsis patients within 12 hours of sepsis onset. Urine was collected from 39 SIRS patients within 4 hours after the end of surgery.\n",
    "Sepsis + SIRs. \n",
    "\n",
    "**GSE168442, GSE168443**, Genomic biomarkers from urine cells indicate a unique metabolomic difference between sepsis and sterile inflammation.\n",
    "Sepsis + SIRs.\n",
    "\n",
    "**GSE112099** Urine was collected from 145 sepsis patients within 12 hours of sepsis onset. Urine was collected from 32 patients set to undergo vascular surgery within 4 hours before surgical incision.Discovery and validation of urinary molecular signature of early sepsis (discovery cohort).\n",
    "Sepsis + vascular surgery . \n",
    "\n",
    "\n",
    "## SIRS and sepsis:(2 datasets)\n",
    "\n",
    "**GSE74224** Discrimination of SIRS from Sepsis in Critically Ill Adults.Retrospective, multi-site study using retrospective physician adjudication as a comparator.\n",
    "Sepsis + SIRs.\n",
    "\n",
    "**GSE63042** An integrated transcriptome and expressed variant analysis of sepsis survival and death.We sequenced peripheral blood RNA of 129 representative subjects with systemic inflammatory response syndrome (SIRS, n=23) or sepsis (infection with SIRS), including 78 sepsis survivors and 28 sepsis nonsurvivors, who had previously undergone plasma proteomic and metabolomic profiling.\n",
    "Sepsis + SIRs . (OUTCOME)\n",
    "\n",
    "\n",
    "## Platelets:\n",
    "\n",
    "**GSE252275** Gene expression analysis in platelets of septic patients. Three group comparison: Control patients, septic patients without thrombocytopenia and septic patients with thrombocytopenia.\n",
    "septic patients with thrombocytopenia + without + controls.\n",
    "\n",
    "**GSE210797** 6 healthy donors and 6 non-viral sepsis patients. A Role for IFITM3 in Regulating Fibrinogen Endocytosis and Platelet Reactivity in Non-Viral Sepsis.\n",
    "Non viral sepsis patients + healthy controls.\n",
    "\n",
    "\n",
    "## Skeletal muscles, cardiac tissues:\n",
    "\n",
    "\n",
    "**GSE13205** Skeletal muscle transcriptome in ICU patients suffering from sepsis induced multiple organ failure, 13 septic samples, 8 controls\n",
    "Septic patients + controls.\n",
    "\n",
    "**GSE79962**, 20 sepsis patients, 11 ischemic heart disease (IHD), 9 nonischemic dilated cardiomyopathy (DCM), 11 non-failing donors.Widespread downregulation of cardiac mitochondrial and sarcomeric genes in patients with sepsis.\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "PRJCA006118-\n",
    "\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.16"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
