{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "d569fede",
   "metadata": {},
   "source": [
    "# Dataset: GSE211210\n",
    "\n",
    "This dataset, identified as GSE211210, explores the molecular mechanisms underlying sepsis pathogenesis, focusing on the role of human ALKBH8. Expression profiling by high-throughput sequencing (RNA-seq).\n",
    "\n",
    "Platform: GPL24676 (Illumina NovaSeq 6000).\n",
    "\n",
    "## Study Design:\n",
    "\n",
    "The study includes 10 patients with acute sepsis and 10 healthy controls. RNA was extracted from blood samples of sepsis patients (n = 5) and healthy controls (n = 5) for RNA-seq analysis.\n",
    "\n",
    "The dataset had already fpkm-normalized data, then we prepared it for downstream analysis. (sepsis_dataGSE211210.csv). (rows = 10 and column = 57)\n",
    "\n",
    "Present genes : 55\n",
    "\n",
    "Missing genes : 0\n",
    "\n",
    "## Random forest model:\n",
    "\n",
    "Then Random forest applied to the dataset that target label considered as factor.(Sepsis and Control)\n",
    "100 randomly split was done for random forest (repeated_splits_metrics21120.csv).\n",
    "Average metrics calculated for all MCC, F1, AUC,... and saved in (average_metrics211210.csv) file. \n",
    "\n",
    "Here is the results:\n",
    "\n",
    "\n",
    "### Average metrics:\n",
    "\n",
    "MCC     0.57\n",
    "F1      0.585858585858586\n",
    "AUC     0.59\n",
    "TPR     0.58\n",
    "TNR     0.58\n",
    "PPV     0.58\n",
    "NPV     0.58\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "### Feature removal analysis:\n",
    "\n",
    "To investigate the importance of each gene in the prediction of the model , in each 100 iterations one feature was removed from the dataset to evaluate the mentioned metrics.(feature_removal_results211210.csv)\n",
    "The plot that is created by results of feature removal:(feature_removal_mcc_plot211210.png)\n",
    "\n",
    "\n",
    "Most important genes based on the plot that we see are the genes that by removing from the dataset cause more drop in MCC value. (**GATA3, CX3CR1, CCL2, BCL2**).\n",
    "\n",
    "Less important genes in the prediction performance of the model are those that their removal does not affect the MCC value significantly. (**HMGB1, IFNB1, NLRP3, S100A12 and CCL25**)\n",
    "\n",
    "\n",
    "### Sanity check:\n",
    "\n",
    "Sanity check has been done on the model to evaluate its performance . By adding more noise step by step to the dataset in each iteration to confirm if the MCC, F1 and AUC value will decline or not.\n",
    "The noise was added to the dataset in 5 different steps as 10%, 20%, 30%, 40% and 50%.(sanity_check_results211210.csv).\n",
    "\n",
    "Here is the plot that shows how the calculated metrics will change by increasing noise on the dataset. (impact_of_noise_on_model_performance211210.png).\n",
    "\n",
    "-------------------------------\n",
    "\n",
    "The problem was many 0 values in some features of the dataset.\n",
    "\n",
    "So we normalized the raw data of the dataset to make sure it's properly normalized. The Deseq2 was used for normalization and then annotation data was used to map genes symbols and filter them and prepare them for downstream analysis. ((sepsis_Deseq_normGSE211210.csv)).But it still had 4 or 5 features with 0 values.\n",
    "\n",
    "The results were the same but there was an issue in results, because the dataset is too small. In the train data we had just 4 sepsis and 4 controls and in test data we had only 1 for each of them which makes the result unreliable.\n",
    "The next step was try to make synthetic remapping methods like ROSE or SMOTE but it didnt work properly because of small dataset and we got these errors each time\n",
    "\n",
    "[1] \"SMOTE balancing failed. Consider upSample or an alternative method.\"\n",
    "\n",
    "[1] \"ROSE balancing failed. Using upSample as fallback.\"\n",
    "\n",
    "Then tried to use an upsample, but results were still the same as before in the number of samples for each label in train and test data.\n",
    "\n",
    "Here is the result after using Upsample for 100 random split:\n",
    "\n",
    "confusion\n",
    "Confusion Matrix and Statistics\n",
    "\n",
    "          Reference\n",
    "Prediction Control Sepsis\n",
    "   Control       1      0\n",
    "   Sepsis        0      1\n",
    "                          \n",
    "                          \n",
    "               Accuracy : 1          \n",
    "                 95% CI : (0.1581, 1)\n",
    "    No Information Rate : 0.5        \n",
    "    P-Value [Acc > NIR] : 0.25       \n",
    "                                     \n",
    "                  Kappa : 1          \n",
    "                                     \n",
    " Mcnemar's Test P-Value : NA         \n",
    "                                     \n",
    "            Sensitivity : 1.0        \n",
    "            Specificity : 1.0        \n",
    "         Pos Pred Value : 1.0        \n",
    "         Neg Pred Value : 1.0        \n",
    "             Prevalence : 0.5        \n",
    "         Detection Rate : 0.5        \n",
    "   Detection Prevalence : 0.5        \n",
    "      Balanced Accuracy : 1.0        \n",
    "                                     \n",
    "       'Positive' Class : Sepsis     \n",
    "                                     \n",
    "> table(train_data_balanced$Label)\n",
    "\n",
    "Control  Sepsis \n",
    "      4       4 \n",
    "      \n",
    "> table(train_data$Label)\n",
    "\n",
    "Control  Sepsis \n",
    "      4       4 \n",
    "      \n",
    "> table(test_data$Label)\n",
    "\n",
    "Control  Sepsis \n",
    "      1       1\n",
    "\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.16"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
