{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "b3618646",
   "metadata": {},
   "source": [
    "# Dataset: GSE100159\n",
    "\n",
    "This dataset represents the development of a cost-effective and practical targeted transcriptional fingerprinting assay (TFA) for monitoring diverse immune responses in humans. The assay leverages a modular transcriptional repertoire framework based on variations in transcript abundance measured on a genome-wide scale in blood samples. The dataset comprises nearly 1000 human subjects across 16 immunologic conditions.\n",
    "\n",
    "The assay is particularly aimed at detecting transcriptional variations that can inform immune responses in conditions such as sepsis, infections, and other immune-related disorders. It focuses on targeted profiling to enable clinically actionable insights.\n",
    "\n",
    "## Overall Design:\n",
    "\n",
    "The dataset includes 47 total samples, divided as follows:(2 recovery from sepsis)\n",
    "\n",
    "Sepsis group: 35 samples\n",
    "\n",
    "Control group: 12 samples\n",
    "\n",
    "This design allows for comparisons between sepsis patients and controls to identify transcriptional differences that could serve as biomarkers or indicators of sepsis.\n",
    "\n",
    "Present genes : 55\n",
    "\n",
    "Missing genes : 0        ELANE exist as “ELA2” and CXCL8 as “IL8”\n",
    "\n",
    "Dataset had been already normalized then data related to only sepsis patients and control ones was extracted. (sepsis_dataGSE100159.csv) it includes (45 rows and 86 columns) because some genes have different probes. 12 sepsis and 33 controls.\n",
    "\n",
    "\n",
    "\n",
    "## Random Forest Model:\n",
    "\n",
    "Here is the result for patients from sepsis patients and healthy ones . Random forest applied to the database 100 times by randomly splitting the dataset.  (repeated_splits_metrics100159.csv)\n",
    "\n",
    "### Average Metrics\n",
    "\n",
    "The average of 100 times repeated for each metric (MCC, F1 score, AUC, TPR, TNR,...) is calculated . (average_metrics100159.csv)\n",
    "\n",
    "\n",
    "MCC      0.949030558123964\n",
    "F1       0.981776223776224\n",
    "AUC      1\n",
    "TPR      0.968333333333333\n",
    "TNR      0.995\n",
    "PPV      0.998571428571429\n",
    "NPV      0.938333333333333\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "### Overall Assessment\n",
    "\n",
    "**MCC** (0.949): Near-perfect overall classification balance.\n",
    "\n",
    "**F1 Score** (0.982): Excellent balance of precision and recall.\n",
    "\n",
    "**AUC** (1): Perfect class separation.\n",
    "\n",
    "**TPR** (0.968) and **TNR** (0.995): High sensitivity and specificity.\n",
    "\n",
    "**PPV** (0.999) and **NPV** (0.938): Accurate predictions for both classes.\n",
    "\n",
    "\n",
    "\n",
    "### Feature removal results:\n",
    "\n",
    "Each feature has been removed from the dataset and all metrics values are calculated to find out the most important genes in true prediction. (feature_removal_results100159.csv)\n",
    "\n",
    "\n",
    "Genes with High Impact (MCC drops significantly):(**FCGR1A.1, BCL2, MYD88, P2RX7, OLFM4.1, HLA-DRA**)\n",
    "Removing these genes causes a sharp decline in MCC, indicating they are critical for the model's classification performance.\n",
    "These are likely the most informative genes for predicting the outcome (e.g., sepsis).\n",
    "\n",
    "\n",
    "Genes with Low Impact (Minimal change in MCC):(**HIF1A.1, PLAUR, NLRP3, IL1R2.1, TLR2, IL1B**)\n",
    "Removing these genes has little to no effect on MCC, suggesting they are less important for the model’s classification task.\n",
    "\n",
    "### Sanity Check:\n",
    "\n",
    "This plot represents the sanity check where different metrics are tracked as noise levels increase.(sanity_check_results100159.csv)\n",
    "\n",
    "\n",
    "Impact of Noise on Metrics:\n",
    "\n",
    "AUC and F1 Score are more robust to noise compared to MCC , which degrade significantly. This suggests the model's ability to rank predictions (AUC) and balance precision/recall (F1 Score) is less affected by noise.\n",
    "\n",
    "MCC drop significantly, indicating that the model's ability to balance between positive and negative cases deteriorates under noisy conditions, especially for identifying negatives (controls).\n",
    "\n",
    "---------------------\n",
    "\n",
    "\n",
    "The important problem in this dataset is the imbalancing of target levels. The train data includes 8 controls and 27 sepsis and test data includes 2 controls and 6 sepsis.\n",
    "The resampling by SMOTE is done to make sure the results are reliable .\n",
    "The ROSE resampling didn't work properly, maybe because of the small size.\n",
    "\n",
    "---------------------\n",
    "\n",
    "Now here is the result for random forest after resampling by SMOTE:\n",
    "\n",
    "### Random Forest Model:\n",
    "\n",
    "Random forest applied to the database 100 times by randomly splitting the dataset and resampling done by SMOTE.  (repeated_splits_metrics_smote100159.csv)\n",
    "\n",
    "### Average Metrics\n",
    "\n",
    "The average of 100 times repeated for each metric (MCC, F1 score, AUC, TPR, TNR,...) is calculated . (average_metrics_smote100159.csv)\n",
    "\n",
    "\n",
    "MCC      0.927886003508775\n",
    "F1       0.973272727272727\n",
    "AUC      1\n",
    "TPR      0.951666666666667\n",
    "TNR      1\n",
    "PPV      1\n",
    "NPV      0.906666666666667\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "There are slight changes in results, but overall results are almost the same.\n",
    "\n",
    "\n",
    "\n",
    "### Feature removal results:\n",
    "\n",
    "Each feature has been removed from the dataset and all metrics values are calculated to find out the most important genes in true prediction. (feature_removal_results_smote100159.csv)\n",
    "\n",
    "\n",
    "Genes with High Impact (MCC drops significantly):(**CD177, CX3CR1.1, MYD88, TNFSF10, ICAM1**)\n",
    "Removing these genes causes a sharp decline in MCC, indicating they are critical for the model's classification performance.\n",
    "These are likely the most informative genes for predicting the outcome (e.g., sepsis).\n",
    "\n",
    "Genes with Low Impact (Minimal change in MCC):(**MAPK14.3, HMGB1, CX3CR1, ELA2, CCR2, MAPK14**)\n",
    "Removing these genes has little to no effect on MCC, suggesting they are less important for the model’s classification task.\n",
    "\n",
    "### Sanity Check:\n",
    "\n",
    "This plot represents the sanity check where different metrics are tracked as noise levels increase, with a dataset balanced using SMOTE (Synthetic Minority Oversampling Technique).(sanity_check_results_smote100159.csv)\n",
    "\n",
    "\n",
    "Impact of Noise on Metrics:\n",
    "\n",
    "AUC and F1 Score are more robust to noise compared to MCC, which degrade significantly. This suggests the model's ability to rank predictions (AUC) and balance precision/recall (F1 Score) is less affected by noise.\n",
    "\n",
    "MCC drops significantly, indicating that the model's ability to balance between positive and negative cases deteriorates under noisy conditions, especially for identifying negatives (controls).\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.16"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
