{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "49b509cd",
   "metadata": {},
   "source": [
    "# Dataset: GSE241238\n",
    "\n",
    "This study aims to analyze and compare the gene expression profiles of Peripheral Blood Mononuclear Cells (PBMCs) collected from healthy individuals and septic patients. PBMCs are a heterogeneous population of immune cells, including lymphocytes (T cells, B cells, NK cells), monocytes, and dendritic cells, which play a crucial role in immune responses.\n",
    "\n",
    "The goal is to identify differentially expressed genes (DEGs) between healthy and sepsis patients, which could help in understanding the molecular mechanisms of sepsis and immune response.\n",
    "\n",
    "Platform Used: Illumina NovaSeq 6000 (GPL24676)\n",
    "\n",
    "he dataset consists of 8 samples, categorized as follows:\n",
    "\n",
    "4 samples from healthy individuals (PBMCs, Healthy)\n",
    "\n",
    "4 samples from septic patients (PBMCs, Sepsis)\n",
    "\n",
    "sepsis_dataGSE241238.csv -> includes 4 sepsis and 4 healthy ones. (8 rows and 57 columns).\n",
    "\n",
    "Present genes: 55\n",
    "\n",
    "Missing genes: 0\n",
    "\n",
    "We used raw data to do Deseq2 normalization by variance stabilizing transformation. And because the number of samples was small we should change the splitting portions to p = 0.7 in random forest.\n",
    "\n",
    "## Random Forest Model:(RF-GSE241238code.R)\n",
    "\n",
    "Random forest applied to the database 100 times by randomly splitting the dataset and resampling done by SMOTE.  (repeated_splits_GSE241238.csv)\n",
    "\n",
    "## Average Metrics\n",
    "\n",
    "The average of 100 times repeated for each metric (MCC, F1 score, AUC, TPR, TNR,...) is calculated . (average_metrics_GSE241238.csv)\n",
    "\n",
    "metrics\n",
    "MCC    1        0\n",
    "F1     1        0\n",
    "AUC    1        0\n",
    "TPR    1        0\n",
    "TNR    1        0\n",
    "PPV    1        0\n",
    "NPV    1        0   \n",
    "\n",
    "\n",
    "\n",
    "\n",
    "## Feature removal results:\n",
    "\n",
    "Each feature has been removed from the dataset and all metrics values are calculated to find out the most important genes in true prediction. (feature_removal_results_GSE241238.csv)\n",
    "\n",
    "\n",
    "\n",
    "Removing these genes causes a sharp decline in MCC, indicating they are critical for the model's classification performance. These are likely the most informative genes for predicting the outcome (e.g., sepsis).\n",
    "\n",
    "## Feature importance results:\n",
    "\n",
    "Feature importance extraction in Random Forest helps us understand which features (genes, biomarkers, etc.) contribute the most to classification performance. we this for each iteration and then make average of all the results.(average_feature_importance_GSE241238.csv)\n",
    "\n",
    "\n",
    "Based on the results of feature extraction we can see some genes such as (**CCl2, IGNG, S100A8, PTX3,**) are the most influential ones in classification of sepsis and healthy patients. So these features reduce impurity (uncertainty) when used in tree splits and with higher Gini scores are more important.\n",
    "\n",
    "\n",
    "## Sanity Check:\n",
    "\n",
    "This plot represents the sanity check where different metrics are tracked as noise levels increase, with a dataset balanced using SMOTE (Synthetic Minority Oversampling Technique).(sanity_check_results_GSE241238.csv)\n",
    "\n",
    "\n",
    "\n",
    "## Mann-Whitney U test:\n",
    "\n",
    "The Mann-Whitney U test (also called the Wilcoxon rank-sum test) is a non-parametric test used to compare two independent groups when the assumptions of normality (required for a t-test) are not met. In our analysis, we want to compare gene expression between sepsis patients and controls. The Mann-Whitney test helps determine if the expression levels are significantly different between these two groups.(Man-W-GSE241238code.R)\n",
    "\n",
    "If the p-value < 0.05, it means that there is a significant difference in gene expression between Sepsis vs. Control for that gene.\n",
    "\n",
    "If p-value â‰¥ 0.05, it means there is no strong evidence of a difference in expression levels.\n",
    "(mann_whitney_results_adjusted_GSE241238.csv)\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.16"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
